{"cells":[{"cell_type":"markdown","source":["### Number 1"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"50109386-ec9a-443b-8488-42682adcd181"}}},{"cell_type":"code","source":["dataDF = spark.read.options(delimiter=',').csv('dbfs:/databricks-datasets/COVID/CORD-19/2020-03-27/metadata.csv', header=True)\ndataDF.show(truncate=False)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"4a7ad402-eee8-49f6-8851-798aa3951fd9"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"+--------+----------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------------+--------------+-----+---------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------+---------------------------+--------------+-------------+---------------+------------------------------+\n|cord_uid|sha                                     |source_x|title                                                                                                                                       |doi           |pmcid|pubmed_id|license|abstract                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |publish_time                                                                                     |authors                                                                                                                                                                                                                                                                          |journal                        |Microsoft Academic Paper ID|WHO #Covidence|has_full_text|full_text_file |url                           |\n+--------+----------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------------+--------------+-----+---------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------+---------------------------+--------------+-------------+---------------+------------------------------+\n|vho70jcx|f056da9c64fbf00a4645ae326e8a4339d015d155|biorxiv |SIANN: Strain Identification by Alignment to Near Neighbors                                                                                 |10.1101/001727|null |null     |biorxiv|Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |2014-01-10                                                                                       |Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy                                                                                                                                                                                                                   |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/001727|\n|i9tbix2v|daf32e013d325a6feb80e83d15aabc64a48fae33|biorxiv |Spatial epidemiology of networked metapopulation: An overview                                                                               |10.1101/003889|null |null     |biorxiv|An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |2014-06-04                                                                                       |Lin WANG; Xiang Li                                                                                                                                                                                                                                                               |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/003889|\n|62gfisc6|f33c6d94b0efaa198f8f3f20e644625fa3fe10d2|biorxiv |Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity     |10.1101/006866|null |null     |biorxiv|Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |2014-07-03                                                                                       |Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden                                                                                                            |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/006866|\n|058r9486|4da8a87e614373d56070ed272487451266dce919|biorxiv |Bayesian mixture analysis for metagenomic community profiling.                                                                              |10.1101/007476|null |null     |biorxiv|Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |2014-07-25                                                                                       |Sofia Morfopoulou; Vincent Plagnol                                                                                                                                                                                                                                               |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/007476|\n|wich35l7|eccef80cfbe078235df22398f195d5db462d8000|biorxiv |Mapping a viral phylogeny onto outbreak trees to improve host transmission inference                                                        |10.1101/010389|null |null     |biorxiv|Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |2014-11-11                                                                                       |Stephen P Velsko; Jonathan E Allen                                                                                                                                                                                                                                               |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/010389|\n|z3tgnzth|c41fdb2efd6d61384a92a84cbba3f8233629a41b|biorxiv |The infant airway microbiome in health and disease impacts later asthma development                                                         |10.1101/012070|null |null     |biorxiv|The nasopharynx (NP) is a reservoir for microbes associated with acute respiratory illnesses (ARI). The development of asthma is initiated during infancy, driven by airway inflammation associated with infections. Here, we report viral and bacterial community profiling of NP aspirates across a birth cohort, capturing all lower respiratory illnesses during their first year. Most infants were initially colonized with Staphylococcus or Corynebacterium before stable colonization with Alloiococcus or Moraxella, with transient incursions of Streptococcus, Moraxella or Haemophilus marking virus-associated ARIs. Our data identify the NP microbiome as a determinant for infection spread to the lower airways, severity of accompanying inflammatory symptoms, and risk for future asthma development. Early asymptomatic colonization with Streptococcus was a strong asthma predictor, and antibiotic usage disrupted asymptomatic colonization patterns.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |2014-12-02                                                                                       |Shu Mei Teo; Danny Mok; Kym Pham; Merci Kusel; Michael Serralha; Niamh Troy; Barbara J Holt; Belinda J Hales; Michael L Walker; Elysia Hollams; Yury H Bochko\n\n*** WARNING: max output size exceeded, skipping output. ***\n\n role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |2016-02-26                                                                                       |Robert J. H. Ross; R. E. Baker; C. A. Yates                                                                                                                                                                                                                                      |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/041509|\n|lcpp5fim|8cfb8dfc2f8b2c6e1d60ff48da02de6670694869|biorxiv |Productive infection of field strains of avian coronavirus infectious bronchitis virus in chicken peripheral blood-derived monocyte         |10.1101/041558|null |null     |biorxiv|The avian coronavirus infectious bronchitis virus (IBV) typically infects the respiratory tract of chickens, but can also spread to other organs. However, the mechanisms of virus dissemination are presently unclear. We show that peripheral blood-derived monocytes/macrophages from chickens (chPBMC) are productively infected by clinical strains of IBV, accompanied by induction of apoptosis. Our data suggest that chPBMCs play a role in the dissemination of IBV, and may be important for viral pathogenesis.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |2016-02-26                                                                                       |Yueting Zhang; Gary Whittaker                                                                                                                                                                                                                                                    |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/041558|\n|gjxumrmm|6499f5c865c3db3bdfc1e969ea940a80d547a5bb|biorxiv |Zika virus outbreak in the Americas: Is Aedes albopictus an overlooked culprit?                                                             |10.1101/044594|null |null     |biorxiv|Summary / AbstractCodon usage patterns of viruses reflect a series of evolutionary changes that enable viruses to shape their survival rates and fitness toward the external environment and, most importantly, their hosts. In the present study, we employed multiple codon usage analysis indices to determine genotype specific codon usage patterns of Zika virus (ZIKV) strains from the current outbreak and those reported previously. Several genotype specific and common codon usage traits were noted in ZIKV coding sequences, indicative of independent evolutionary origins from a common ancestor. The overall influence of natural selection was found to be more profound than that of mutation pressure and acting on specific set of viral genes belonging to ZIKV strains of Asian genotype from the recent outbreak. Furthermore, an interplay of codon adaptation and deoptimization have been observed in ZIKV genomes. The collective findings of codon analysis in association with the geographical data of Aedes populations in the Americas suggests that ZIKV have evolved a dynamic set of codon usage patterns in order to maintain a successful replication and transmission chain within multiple hosts and vectors.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |2016-03-30                                                                                       |Azeem Mehmood Butt; Izza Nasrullah; Raheel Qamar; Yigang Tong                                                                                                                                                                                                                    |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/044594|\n|csokkcqq|a59906b732bf4a489e282c3e4f499d4166c622e7|biorxiv |Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim                          |10.1101/046532|null |null     |biorxiv|AbstractBackgroundIn silico bacterial, viral, and human truth datasets were generated to evaluate available metagenomics algorithms. Sequenced datasets include background organisms, creating ambiguity in the true source organism for each read. Bacterial and viral datasets were created with even and staggered coverage to evaluate organism identification, read mapping, and gene identification capabilities of available algorithms. These truth datasets are provided as a resource for the development and refinement of metagenomic algorithms. Algorithm performance on these truth datasets can inform decision makers on strengths and weaknesses of available algorithms and how the results may be best leveraged for bacterial and viral organism identification and characterization.Source organisms were selected to mirror communities described in the Human Microbiome Project as well as the emerging pathogens listed by the National Institute of Allergy and Infectious Diseases. The six in silico datasets were used to evaluate the performance of six leading metagenomics algorithms: MetaScope, Kraken, LMAT, MetaPhlAn, MetaCV, and MetaPhyler.ResultsAlgorithms were evaluated on runtime, true positive organisms identified to the genus and species levels, false positive organisms identified to genus and species level, read mapping, relative abundance estimation, and gene calling. No algorithm out performed the others in all categories, and the algorithm or algorithms of choice strongly depends on analysis goals. MetaPhlAn excels for bacteria and LMAT for viruses. The algorithms were ranked by overall performance using a normalized weighted sum of the above metrics, and MetaScope emerged as the overall winner, followed by Kraken and LMAT.ConclusionsSimulated FASTQ datasets with well-characterized truth data about microbial community composition reveal numerous insights about the relative strengths and weaknesses of the metagenomics algorithms evaluated. The simulated datasets are available to download from the Sequence Read Archive (SRP062063).|2016-03-31                                                                                       |Anna Shcherbina; Darrell O. Ricke; Nelson Chiu                                                                                                                                                                                                                                   |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/046532|\n|08mep8hm|99dab788e984601ee4efcd1138926caf10c00dcf|biorxiv |Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension                                                          |10.1101/048801|null |null     |biorxiv|Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (fs) and monovalent salt concentrations (Cs). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log Cm)α with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |2016-04-15                                                                                       |Naoto Hori; Natalia A. Denesyuk; D. Thirumalai                                                                                                                                                                                                                                   |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/048801|\n|ejqq2wmi|97339c9d365437d54ea33be1a1b634d35f460208|biorxiv |Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study                                                                |10.1101/053421|null |null     |biorxiv|BackgroundThe ongoing yellow fever (YF) epidemic in Angola strains the global vaccine supply, prompting WHO to adopt dose sparing for its vaccination campaign in Kinshasa in July-August 2016. Although a 5-fold fractional-dose vaccine is similar to standard-dose vaccine in safety and immunogenicity, efficacy is untested. There is an urgent need to ensure the robustness of fractional-dose vaccination by elucidating the conditions under which dose fractionation would reduce transmission.MethodsWe estimate the effective reproductive number for YF in Angola using disease natural history and case report data. With simple mathematical models of YF transmission, we calculate the infection attack rate (IAR, the proportion of population infected over the course of an epidemic) under varying levels of transmissibility and five-fold fractional-dose vaccine efficacy for two vaccination scenarios: (i) random vaccination in a hypothetical population that is completely susceptible; (ii) the Kinshasa vaccination campaign in July-August 2016 with different age cutoff for fractional-dose vaccines.FindingsWe estimate the effective reproductive number early in the Angola outbreak was between 5·2 and 7·1. If vaccine action is all-or-nothing (i.e. a proportion VE of vaccinees receives complete and the remainder receive no protection), n-fold fractionation can dramatically reduce IAR as long as efficacy VE exceeds 1/n. This benefit threshold becomes more stringent if vaccine action is leaky (i.e. the susceptibility of each vaccinee is reduced by a factor that is equal to the vaccine efficacy VE). The age cutoff for fractional-dose vaccines chosen by the WHO for the Kinshasa vaccination campaign (namely, 2 years) provides the largest reduction in IAR if the efficacy of five-fold fractional-dose vaccines exceeds 20%.InterpretationDose fractionation is a very effective strategy for reducing infection attack rate that would be robust with a large margin for error in case fractional-dose VE is lower than expected.FundingNIH-MIDAS, HMRF-Hong Kong    |2016-05-14                                                                                       |Joseph T. Wu; Corey M. Peak; Gabriel M. Leung; Marc Lipsitch                                                                                                                                                                                                                     |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/053421|\n|2eh0bhee|1c0b25c8e3c89243fd8797e2e964e12689bc8556|biorxiv |50-valent inactivated rhinovirus vaccine is broadly immunogenic in rhesus macaques                                                          |10.1101/053967|null |null     |biorxiv|AbstractAs the predominant etiological agent of the common cold, human rhinovirus (HRV) is the leading cause of human infectious disease. Early studies showed monovalent formalin-inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated with protection. However, co-circulation of many HRV types discouraged further vaccine efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, we show that serum nAb against many rhinovirus types can be induced by polyvalent, inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient quantity of input antigens, and valency was not a major factor for potency or breadth of the response. We for the first time generated a vaccine capable of inducing nAb responses to numerous and diverse HRV types.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |2016-05-17                                                                                       |Sujin Lee; Minh Trang Nguyen; Michael G. Currier; Joe B. Jenkins; Elizabeth A. Strobert; Adriana E. Kajon; Ranjna Madan-Lala; Yury A. Bochkov; James E. Gern; Krishnendu Roy; Xiaoyan Lu; Dean D. Erdman; Paul Spearman; Martin L. Moore                                         |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/053967|\n|76i8yvb4|d1b26388670660f66249b55ece2beccee4c18d1a|biorxiv |Evidence for reassortment of highly divergent novel rotaviruses from bats in Cameroon, without evidence for human interspecies transmissions|10.1101/054072|null |null     |biorxiv|AbstractBats are an important reservoir for pathogenic human respiratory and hemorrhagic viruses but only little is known about bat viruses causing gastroenteritis in humans, including rotavirus A strains (RVA). Only three RVA strains have been reported in bats in Kenya (straw-colored fruit bat) and in China (lesser horseshoe and a stoliczka’s trident bat), being highly divergent from each other. To further elucidate the potential of bat RVAs to cause gastroenteritis in humans we started by investigating the genetic diversity of RVAs in fecal samples from 87 straw-colored fruit bats living in close contact with humans in Cameroon using metagenomics. Five samples contained significant numbers of RVA Illumina reads, sufficient to obtain their (near) complete genomes. A single RVA strain showed a close phylogenetic relationship with the Kenyan bat RVA strain in six gene segments, including VP7 (G25), whereas the other gene segments represented novel genotypes as ratified by the RCWG. The 4 other RVA strains were highly divergent from known strains (but very similar among each other) possessing all novel genotypes. Only the VP7 and VP4 genes showed a significant variability representing multiple novel G and P genotypes, indicating the frequent occurrence of reassortment events.Comparing these bat RVA strains with currently used human RVA screening primers indicated that several of the novel VP7 and VP4 segments would not be detected in routine epidemiological screening studies. Therefore, novel VP6 based screening primers matching both human and bat RVAs were developed and used to screen samples from 25 infants with gastroenteritis living in close proximity with the studied bat population. Although RVA infections were identified in 36% of the infants, Sanger sequencing did not indicate evidence of interspecies transmissions.This study identified multiple novel bat RVA strains, but further epidemiological studies in humans will have to assess if these viruses have the potential to cause gastroenteritis in humans.             |2016-05-18                                                                                       |Claude Kwe Yinda; Mark Zeller; Nádia Conceição-Neto; Piet Maes; Ward Deboutte; Leen Beller; Elisabeth Heylen; Stephen Mbigha Ghogomu; Marc Van Ranst; Jelle Matthijnssens                                                                                                        |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/054072|\n+--------+----------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------------+--------------+-----+---------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------+---------------------------+--------------+-------------+---------------+------------------------------+\nonly showing top 20 rows\n\n","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["+--------+----------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------------+--------------+-----+---------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------+---------------------------+--------------+-------------+---------------+------------------------------+\n|cord_uid|sha                                     |source_x|title                                                                                                                                       |doi           |pmcid|pubmed_id|license|abstract                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |publish_time                                                                                     |authors                                                                                                                                                                                                                                                                          |journal                        |Microsoft Academic Paper ID|WHO #Covidence|has_full_text|full_text_file |url                           |\n+--------+----------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------------+--------------+-----+---------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------+---------------------------+--------------+-------------+---------------+------------------------------+\n|vho70jcx|f056da9c64fbf00a4645ae326e8a4339d015d155|biorxiv |SIANN: Strain Identification by Alignment to Near Neighbors                                                                                 |10.1101/001727|null |null     |biorxiv|Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |2014-01-10                                                                                       |Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy                                                                                                                                                                                                                   |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/001727|\n|i9tbix2v|daf32e013d325a6feb80e83d15aabc64a48fae33|biorxiv |Spatial epidemiology of networked metapopulation: An overview                                                                               |10.1101/003889|null |null     |biorxiv|An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |2014-06-04                                                                                       |Lin WANG; Xiang Li                                                                                                                                                                                                                                                               |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/003889|\n|62gfisc6|f33c6d94b0efaa198f8f3f20e644625fa3fe10d2|biorxiv |Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity     |10.1101/006866|null |null     |biorxiv|Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |2014-07-03                                                                                       |Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden                                                                                                            |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/006866|\n|058r9486|4da8a87e614373d56070ed272487451266dce919|biorxiv |Bayesian mixture analysis for metagenomic community profiling.                                                                              |10.1101/007476|null |null     |biorxiv|Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |2014-07-25                                                                                       |Sofia Morfopoulou; Vincent Plagnol                                                                                                                                                                                                                                               |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/007476|\n|wich35l7|eccef80cfbe078235df22398f195d5db462d8000|biorxiv |Mapping a viral phylogeny onto outbreak trees to improve host transmission inference                                                        |10.1101/010389|null |null     |biorxiv|Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |2014-11-11                                                                                       |Stephen P Velsko; Jonathan E Allen                                                                                                                                                                                                                                               |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/010389|\n|z3tgnzth|c41fdb2efd6d61384a92a84cbba3f8233629a41b|biorxiv |The infant airway microbiome in health and disease impacts later asthma development                                                         |10.1101/012070|null |null     |biorxiv|The nasopharynx (NP) is a reservoir for microbes associated with acute respiratory illnesses (ARI). The development of asthma is initiated during infancy, driven by airway inflammation associated with infections. Here, we report viral and bacterial community profiling of NP aspirates across a birth cohort, capturing all lower respiratory illnesses during their first year. Most infants were initially colonized with Staphylococcus or Corynebacterium before stable colonization with Alloiococcus or Moraxella, with transient incursions of Streptococcus, Moraxella or Haemophilus marking virus-associated ARIs. Our data identify the NP microbiome as a determinant for infection spread to the lower airways, severity of accompanying inflammatory symptoms, and risk for future asthma development. Early asymptomatic colonization with Streptococcus was a strong asthma predictor, and antibiotic usage disrupted asymptomatic colonization patterns.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |2014-12-02                                                                                       |Shu Mei Teo; Danny Mok; Kym Pham; Merci Kusel; Michael Serralha; Niamh Troy; Barbara J Holt; Belinda J Hales; Michael L Walker; Elysia Hollams; Yury H Bochko\n\n*** WARNING: max output size exceeded, skipping output. ***\n\n role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |2016-02-26                                                                                       |Robert J. H. Ross; R. E. Baker; C. A. Yates                                                                                                                                                                                                                                      |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/041509|\n|lcpp5fim|8cfb8dfc2f8b2c6e1d60ff48da02de6670694869|biorxiv |Productive infection of field strains of avian coronavirus infectious bronchitis virus in chicken peripheral blood-derived monocyte         |10.1101/041558|null |null     |biorxiv|The avian coronavirus infectious bronchitis virus (IBV) typically infects the respiratory tract of chickens, but can also spread to other organs. However, the mechanisms of virus dissemination are presently unclear. We show that peripheral blood-derived monocytes/macrophages from chickens (chPBMC) are productively infected by clinical strains of IBV, accompanied by induction of apoptosis. Our data suggest that chPBMCs play a role in the dissemination of IBV, and may be important for viral pathogenesis.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |2016-02-26                                                                                       |Yueting Zhang; Gary Whittaker                                                                                                                                                                                                                                                    |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/041558|\n|gjxumrmm|6499f5c865c3db3bdfc1e969ea940a80d547a5bb|biorxiv |Zika virus outbreak in the Americas: Is Aedes albopictus an overlooked culprit?                                                             |10.1101/044594|null |null     |biorxiv|Summary / AbstractCodon usage patterns of viruses reflect a series of evolutionary changes that enable viruses to shape their survival rates and fitness toward the external environment and, most importantly, their hosts. In the present study, we employed multiple codon usage analysis indices to determine genotype specific codon usage patterns of Zika virus (ZIKV) strains from the current outbreak and those reported previously. Several genotype specific and common codon usage traits were noted in ZIKV coding sequences, indicative of independent evolutionary origins from a common ancestor. The overall influence of natural selection was found to be more profound than that of mutation pressure and acting on specific set of viral genes belonging to ZIKV strains of Asian genotype from the recent outbreak. Furthermore, an interplay of codon adaptation and deoptimization have been observed in ZIKV genomes. The collective findings of codon analysis in association with the geographical data of Aedes populations in the Americas suggests that ZIKV have evolved a dynamic set of codon usage patterns in order to maintain a successful replication and transmission chain within multiple hosts and vectors.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |2016-03-30                                                                                       |Azeem Mehmood Butt; Izza Nasrullah; Raheel Qamar; Yigang Tong                                                                                                                                                                                                                    |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/044594|\n|csokkcqq|a59906b732bf4a489e282c3e4f499d4166c622e7|biorxiv |Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim                          |10.1101/046532|null |null     |biorxiv|AbstractBackgroundIn silico bacterial, viral, and human truth datasets were generated to evaluate available metagenomics algorithms. Sequenced datasets include background organisms, creating ambiguity in the true source organism for each read. Bacterial and viral datasets were created with even and staggered coverage to evaluate organism identification, read mapping, and gene identification capabilities of available algorithms. These truth datasets are provided as a resource for the development and refinement of metagenomic algorithms. Algorithm performance on these truth datasets can inform decision makers on strengths and weaknesses of available algorithms and how the results may be best leveraged for bacterial and viral organism identification and characterization.Source organisms were selected to mirror communities described in the Human Microbiome Project as well as the emerging pathogens listed by the National Institute of Allergy and Infectious Diseases. The six in silico datasets were used to evaluate the performance of six leading metagenomics algorithms: MetaScope, Kraken, LMAT, MetaPhlAn, MetaCV, and MetaPhyler.ResultsAlgorithms were evaluated on runtime, true positive organisms identified to the genus and species levels, false positive organisms identified to genus and species level, read mapping, relative abundance estimation, and gene calling. No algorithm out performed the others in all categories, and the algorithm or algorithms of choice strongly depends on analysis goals. MetaPhlAn excels for bacteria and LMAT for viruses. The algorithms were ranked by overall performance using a normalized weighted sum of the above metrics, and MetaScope emerged as the overall winner, followed by Kraken and LMAT.ConclusionsSimulated FASTQ datasets with well-characterized truth data about microbial community composition reveal numerous insights about the relative strengths and weaknesses of the metagenomics algorithms evaluated. The simulated datasets are available to download from the Sequence Read Archive (SRP062063).|2016-03-31                                                                                       |Anna Shcherbina; Darrell O. Ricke; Nelson Chiu                                                                                                                                                                                                                                   |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/046532|\n|08mep8hm|99dab788e984601ee4efcd1138926caf10c00dcf|biorxiv |Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension                                                          |10.1101/048801|null |null     |biorxiv|Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (fs) and monovalent salt concentrations (Cs). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log Cm)α with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |2016-04-15                                                                                       |Naoto Hori; Natalia A. Denesyuk; D. Thirumalai                                                                                                                                                                                                                                   |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/048801|\n|ejqq2wmi|97339c9d365437d54ea33be1a1b634d35f460208|biorxiv |Fractional Dosing of Yellow Fever Vaccine to Extend Supply: A Modeling Study                                                                |10.1101/053421|null |null     |biorxiv|BackgroundThe ongoing yellow fever (YF) epidemic in Angola strains the global vaccine supply, prompting WHO to adopt dose sparing for its vaccination campaign in Kinshasa in July-August 2016. Although a 5-fold fractional-dose vaccine is similar to standard-dose vaccine in safety and immunogenicity, efficacy is untested. There is an urgent need to ensure the robustness of fractional-dose vaccination by elucidating the conditions under which dose fractionation would reduce transmission.MethodsWe estimate the effective reproductive number for YF in Angola using disease natural history and case report data. With simple mathematical models of YF transmission, we calculate the infection attack rate (IAR, the proportion of population infected over the course of an epidemic) under varying levels of transmissibility and five-fold fractional-dose vaccine efficacy for two vaccination scenarios: (i) random vaccination in a hypothetical population that is completely susceptible; (ii) the Kinshasa vaccination campaign in July-August 2016 with different age cutoff for fractional-dose vaccines.FindingsWe estimate the effective reproductive number early in the Angola outbreak was between 5·2 and 7·1. If vaccine action is all-or-nothing (i.e. a proportion VE of vaccinees receives complete and the remainder receive no protection), n-fold fractionation can dramatically reduce IAR as long as efficacy VE exceeds 1/n. This benefit threshold becomes more stringent if vaccine action is leaky (i.e. the susceptibility of each vaccinee is reduced by a factor that is equal to the vaccine efficacy VE). The age cutoff for fractional-dose vaccines chosen by the WHO for the Kinshasa vaccination campaign (namely, 2 years) provides the largest reduction in IAR if the efficacy of five-fold fractional-dose vaccines exceeds 20%.InterpretationDose fractionation is a very effective strategy for reducing infection attack rate that would be robust with a large margin for error in case fractional-dose VE is lower than expected.FundingNIH-MIDAS, HMRF-Hong Kong    |2016-05-14                                                                                       |Joseph T. Wu; Corey M. Peak; Gabriel M. Leung; Marc Lipsitch                                                                                                                                                                                                                     |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/053421|\n|2eh0bhee|1c0b25c8e3c89243fd8797e2e964e12689bc8556|biorxiv |50-valent inactivated rhinovirus vaccine is broadly immunogenic in rhesus macaques                                                          |10.1101/053967|null |null     |biorxiv|AbstractAs the predominant etiological agent of the common cold, human rhinovirus (HRV) is the leading cause of human infectious disease. Early studies showed monovalent formalin-inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated with protection. However, co-circulation of many HRV types discouraged further vaccine efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, we show that serum nAb against many rhinovirus types can be induced by polyvalent, inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient quantity of input antigens, and valency was not a major factor for potency or breadth of the response. We for the first time generated a vaccine capable of inducing nAb responses to numerous and diverse HRV types.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |2016-05-17                                                                                       |Sujin Lee; Minh Trang Nguyen; Michael G. Currier; Joe B. Jenkins; Elizabeth A. Strobert; Adriana E. Kajon; Ranjna Madan-Lala; Yury A. Bochkov; James E. Gern; Krishnendu Roy; Xiaoyan Lu; Dean D. Erdman; Paul Spearman; Martin L. Moore                                         |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/053967|\n|76i8yvb4|d1b26388670660f66249b55ece2beccee4c18d1a|biorxiv |Evidence for reassortment of highly divergent novel rotaviruses from bats in Cameroon, without evidence for human interspecies transmissions|10.1101/054072|null |null     |biorxiv|AbstractBats are an important reservoir for pathogenic human respiratory and hemorrhagic viruses but only little is known about bat viruses causing gastroenteritis in humans, including rotavirus A strains (RVA). Only three RVA strains have been reported in bats in Kenya (straw-colored fruit bat) and in China (lesser horseshoe and a stoliczka’s trident bat), being highly divergent from each other. To further elucidate the potential of bat RVAs to cause gastroenteritis in humans we started by investigating the genetic diversity of RVAs in fecal samples from 87 straw-colored fruit bats living in close contact with humans in Cameroon using metagenomics. Five samples contained significant numbers of RVA Illumina reads, sufficient to obtain their (near) complete genomes. A single RVA strain showed a close phylogenetic relationship with the Kenyan bat RVA strain in six gene segments, including VP7 (G25), whereas the other gene segments represented novel genotypes as ratified by the RCWG. The 4 other RVA strains were highly divergent from known strains (but very similar among each other) possessing all novel genotypes. Only the VP7 and VP4 genes showed a significant variability representing multiple novel G and P genotypes, indicating the frequent occurrence of reassortment events.Comparing these bat RVA strains with currently used human RVA screening primers indicated that several of the novel VP7 and VP4 segments would not be detected in routine epidemiological screening studies. Therefore, novel VP6 based screening primers matching both human and bat RVAs were developed and used to screen samples from 25 infants with gastroenteritis living in close proximity with the studied bat population. Although RVA infections were identified in 36% of the infants, Sanger sequencing did not indicate evidence of interspecies transmissions.This study identified multiple novel bat RVA strains, but further epidemiological studies in humans will have to assess if these viruses have the potential to cause gastroenteritis in humans.             |2016-05-18                                                                                       |Claude Kwe Yinda; Mark Zeller; Nádia Conceição-Neto; Piet Maes; Ward Deboutte; Leen Beller; Elisabeth Heylen; Stephen Mbigha Ghogomu; Marc Van Ranst; Jelle Matthijnssens                                                                                                        |null                           |null                       |null          |True         |biorxiv_medrxiv|https://doi.org/10.1101/054072|\n+--------+----------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------------+--------------+-----+---------+-------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------+---------------------------+--------------+-------------+---------------+------------------------------+\nonly showing top 20 rows\n\n"]}}],"execution_count":0},{"cell_type":"code","source":["dataDF.printSchema()"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"94337d30-d090-4700-85e0-75d17faea619"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"root\n |-- cord_uid: string (nullable = true)\n |-- sha: string (nullable = true)\n |-- source_x: string (nullable = true)\n |-- title: string (nullable = true)\n |-- doi: string (nullable = true)\n |-- pmcid: string (nullable = true)\n |-- pubmed_id: string (nullable = true)\n |-- license: string (nullable = true)\n |-- abstract: string (nullable = true)\n |-- publish_time: string (nullable = true)\n |-- authors: string (nullable = true)\n |-- journal: string (nullable = true)\n |-- Microsoft Academic Paper ID: string (nullable = true)\n |-- WHO #Covidence: string (nullable = true)\n |-- has_full_text: string (nullable = true)\n |-- full_text_file: string (nullable = true)\n |-- url: string (nullable = true)\n\n","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["root\n |-- cord_uid: string (nullable = true)\n |-- sha: string (nullable = true)\n |-- source_x: string (nullable = true)\n |-- title: string (nullable = true)\n |-- doi: string (nullable = true)\n |-- pmcid: string (nullable = true)\n |-- pubmed_id: string (nullable = true)\n |-- license: string (nullable = true)\n |-- abstract: string (nullable = true)\n |-- publish_time: string (nullable = true)\n |-- authors: string (nullable = true)\n |-- journal: string (nullable = true)\n |-- Microsoft Academic Paper ID: string (nullable = true)\n |-- WHO #Covidence: string (nullable = true)\n |-- has_full_text: string (nullable = true)\n |-- full_text_file: string (nullable = true)\n |-- url: string (nullable = true)\n\n"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 2"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"b3239241-9290-4c14-8333-f88eb53f3d7b"}}},{"cell_type":"code","source":["dataRDD = dataDF.rdd\ndataRDD.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"415b8f2b-4db3-4675-b748-548ee47a0c0f"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[3]: [Row(cord_uid='vho70jcx', sha='f056da9c64fbf00a4645ae326e8a4339d015d155', source_x='biorxiv', title='SIANN: Strain Identification by Alignment to Near Neighbors', doi='10.1101/001727', pmcid=None, pubmed_id=None, license='biorxiv', abstract='Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.', publish_time='2014-01-10', authors='Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/001727'),\n Row(cord_uid='i9tbix2v', sha='daf32e013d325a6feb80e83d15aabc64a48fae33', source_x='biorxiv', title='Spatial epidemiology of networked metapopulation: An overview', doi='10.1101/003889', pmcid=None, pubmed_id=None, license='biorxiv', abstract='An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.', publish_time='2014-06-04', authors='Lin WANG; Xiang Li', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/003889'),\n Row(cord_uid='62gfisc6', sha='f33c6d94b0efaa198f8f3f20e644625fa3fe10d2', source_x='biorxiv', title='Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity', doi='10.1101/006866', pmcid=None, pubmed_id=None, license='biorxiv', abstract='Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.', publish_time='2014-07-03', authors='Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/006866'),\n Row(cord_uid='058r9486', sha='4da8a87e614373d56070ed272487451266dce919', source_x='biorxiv', title='Bayesian mixture analysis for metagenomic community profiling.', doi='10.1101/007476', pmcid=None, pubmed_id=None, license='biorxiv', abstract='Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix', publish_time='2014-07-25', authors='Sofia Morfopoulou; Vincent Plagnol', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/007476'),\n Row(cord_uid='wich35l7', sha='eccef80cfbe078235df22398f195d5db462d8000', source_x='biorxiv', title='Mapping a viral phylogeny onto outbreak trees to improve host transmission inference', doi='10.1101/010389', pmcid=None, pubmed_id=None, license='biorxiv', abstract=\"Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.\", publish_time='2014-11-11', authors='Stephen P Velsko; Jonathan E Allen', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/010389')]","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[3]: [Row(cord_uid='vho70jcx', sha='f056da9c64fbf00a4645ae326e8a4339d015d155', source_x='biorxiv', title='SIANN: Strain Identification by Alignment to Near Neighbors', doi='10.1101/001727', pmcid=None, pubmed_id=None, license='biorxiv', abstract='Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.', publish_time='2014-01-10', authors='Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/001727'),\n Row(cord_uid='i9tbix2v', sha='daf32e013d325a6feb80e83d15aabc64a48fae33', source_x='biorxiv', title='Spatial epidemiology of networked metapopulation: An overview', doi='10.1101/003889', pmcid=None, pubmed_id=None, license='biorxiv', abstract='An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.', publish_time='2014-06-04', authors='Lin WANG; Xiang Li', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/003889'),\n Row(cord_uid='62gfisc6', sha='f33c6d94b0efaa198f8f3f20e644625fa3fe10d2', source_x='biorxiv', title='Sequencing of the human IG light chain loci from a hydatidiform mole BAC library reveals locus-specific signatures of genetic diversity', doi='10.1101/006866', pmcid=None, pubmed_id=None, license='biorxiv', abstract='Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.', publish_time='2014-07-03', authors='Corey T Watson; Karyn Meltz Steinberg; Tina A Graves-Lindsay; Rene L Warren; Maika Malig; Jacqueline E Schein; Richard K Wilson; Rob Holt; Evan Eichler; Felix Breden', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/006866'),\n Row(cord_uid='058r9486', sha='4da8a87e614373d56070ed272487451266dce919', source_x='biorxiv', title='Bayesian mixture analysis for metagenomic community profiling.', doi='10.1101/007476', pmcid=None, pubmed_id=None, license='biorxiv', abstract='Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix', publish_time='2014-07-25', authors='Sofia Morfopoulou; Vincent Plagnol', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/007476'),\n Row(cord_uid='wich35l7', sha='eccef80cfbe078235df22398f195d5db462d8000', source_x='biorxiv', title='Mapping a viral phylogeny onto outbreak trees to improve host transmission inference', doi='10.1101/010389', pmcid=None, pubmed_id=None, license='biorxiv', abstract=\"Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.\", publish_time='2014-11-11', authors='Stephen P Velsko; Jonathan E Allen', journal=None, Microsoft Academic Paper ID=None, WHO #Covidence=None, has_full_text='True', full_text_file='biorxiv_medrxiv', url='https://doi.org/10.1101/010389')]"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 3"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"4b94c221-5a09-443b-8899-42011fa5e79f"}}},{"cell_type":"code","source":["pairRDD = dataRDD.map(lambda x: (x[0], x[8]))\npairRDD.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"bce11c8f-15d5-4387-a350-34220cd0944b"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[4]: [('vho70jcx',\n  'Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.'),\n ('i9tbix2v',\n  'An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.'),\n ('62gfisc6',\n  'Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.'),\n ('058r9486',\n  'Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix'),\n ('wich35l7',\n  \"Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.\")]","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[4]: [('vho70jcx',\n  'Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.'),\n ('i9tbix2v',\n  'An emerging disease is one infectious epidemic caused by a newly transmissible pathogen, which has either appeared for the first time or already existed in human populations, having the capacity to increase rapidly in incidence as well as geographic range. Adapting to human immune system, emerging diseases may trigger large-scale pandemic spreading, such as the transnational spreading of SARS, the global outbreak of A(H1N1), and the recent potential invasion of avian influenza A(H7N9). To study the dynamics mediating the transmission of emerging diseases, spatial epidemiology of networked metapopulation provides a valuable modeling framework, which takes spatially distributed factors into consideration. This review elaborates the latest progresses on the spatial metapopulation dynamics, discusses empirical and theoretical findings that verify the validity of networked metapopulations, and the application in evaluating the effectiveness of disease intervention strategies as well.'),\n ('62gfisc6',\n  'Germline variation at immunoglobulin gene (IG) loci is critical for pathogen-mediated immunity, but establishing complete reference sequences in these regions is problematic because of segmental duplications and somatically rearranged source DNA. We sequenced BAC clones from the essentially haploid hydatidiform mole, CHM1, across the light chain IG loci, kappa (IGK) and lambda (IGL), creating single haplotype representations of these regions. The IGL haplotype is 1.25Mb of contiguous sequence with four novel V gene and one novel C gene alleles and an 11.9kbp insertion. The IGK haplotype consists of two 644kbp proximal and 466kbp distal contigs separated by a gap also present in the reference genome sequence. Our effort added an additional 49kbp of unique sequence extending into this gap. The IGK haplotype contains six novel V gene and one novel J gene alleles and a 16.7kbp region with increased sequence identity between the two IGK contigs, exhibiting signatures of interlocus gene conversion. Our data facilitated the first comparison of nucleotide diversity between the light and IG heavy (IGH) chain haplotypes within a single genome, revealing a three to six fold enrichment in the IGH locus, supporting the theory that the heavy chain may be more important in determining antigenic specificity.'),\n ('058r9486',\n  'Deep sequencing of clinical samples is now an established tool for the detection of infectious pathogens, with direct medical applications. The large amount of data generated provides an opportunity to detect species even at very low levels, provided that computational tools can effectively interpret potentially complex metagenomic mixtures. Data interpretation is complicated by the fact that short sequencing reads can match multiple organisms and by the lack of completeness of existing databases, in particular for viral pathogens. This interpretation problem can be formulated statistically as a mixture model, where the species of origin of each read is missing, but the complete knowledge of all species present in the mixture helps with the individual reads assignment. Several analytical tools have been proposed to approximately solve this computational problem. Here, we show that the use of parallel Monte Carlo Markov chains (MCMC) for the exploration of the species space enables the identification of the set of species most likely to contribute to the mixture. The added accuracy comes at a cost of increased computation time. Our approach is useful for solving complex mixtures involving several related species. We designed our method specifically for the analysis of deep transcriptome sequencing datasets and with a particular focus on viral pathogen detection, but the principles are applicable more generally to all types of metagenomics mixtures. The work is implemented as a user friendly R package, available from CRAN: http://cran.r-project.org/web/packages/metaMix'),\n ('wich35l7',\n  \"Developing methods to reconstruct transmission histories for viral outbreaks could provide critical information to support locating sources of disease transmission. Phylogenetic methods used to measure the degree of relatedness among sequenced viral samples have proven useful in identifying potential outbreak sources. The complex nature of infectious disease, however, makes it difficult to assign a rigorously defined quantitative confidence value assessing the likelihood of a true direct transmission event using genetic data alone. A new method is presented to calculate a confidence value assessing the likelihood of a transmission event using both phylogenetic inference and limited knowledge of incubation and infectious duration times. The method is applied to simulations of a foot and mouth disease (FMD) outbreak to demonstrate how the combination of both phylogenetic and epidemiology data can be used to strengthen the assessment of the likelihood of direct transmission over methods using just phylogenetic data or infection timing data alone. The method is applied to a previous FMD outbreak to identify areas where over confidence in previously inferred direct transmission may exist. Combining knowledge from viral evolution and epidemiology within a single integrated transmission inference framework is an important approach to assess the potential likelihood of transmission events and makes clear how specific features of a virus' spread through the course of an outbreak will directly determine the potential for confidence in inferred host transmission links.\")]"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 4"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"fba066a4-143b-47c5-b6ee-1082f0dfb664"}}},{"cell_type":"code","source":["pairRDD = pairRDD.filter(lambda x: x[1] != None)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"c1a6a598-9178-4da1-98c5-f54850d03043"}},"outputs":[],"execution_count":0},{"cell_type":"code","source":["wordPairRDD = pairRDD.map(lambda j: (j[0],j[1].split(' ')))\nwordPairRDD.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"27d0772d-028c-458b-aca0-2aaef1ef3017"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[7]: [('vho70jcx',\n  ['Next-generation',\n   'sequencing',\n   'is',\n   'increasingly',\n   'being',\n   'used',\n   'to',\n   'study',\n   'samples',\n   'composed',\n   'of',\n   'mixtures',\n   'of',\n   'organisms,',\n   'such',\n   'as',\n   'in',\n   'clinical',\n   'applications',\n   'where',\n   'the',\n   'presence',\n   'of',\n   'a',\n   'pathogen',\n   'at',\n   'very',\n   'low',\n   'abundance',\n   'may',\n   'be',\n   'highly',\n   'important.',\n   'We',\n   'present',\n   'an',\n   'analytical',\n   'method',\n   '(SIANN:',\n   'Strain',\n   'Identification',\n   'by',\n   'Alignment',\n   'to',\n   'Near',\n   'Neighbors)',\n   'specifically',\n   'designed',\n   'to',\n   'rapidly',\n   'detect',\n   'a',\n   'set',\n   'of',\n   'target',\n   'organisms',\n   'in',\n   'mixed',\n   'samples',\n   'that',\n   'achieves',\n   'a',\n   'high',\n   'degree',\n   'of',\n   'species-',\n   'and',\n   'strain-specificity',\n   'by',\n   'aligning',\n   'short',\n   'sequence',\n   'reads',\n   'to',\n   'the',\n   'genomes',\n   'of',\n   'near',\n   'neighbor',\n   'organisms,',\n   'as',\n   'well',\n   'as',\n   'that',\n   'of',\n   'the',\n   'target.',\n   'Empirical',\n   'benchmarking',\n   'alongside',\n   'the',\n   'current',\n   'state-of-the-art',\n   'methods',\n   'shows',\n   'an',\n   'extremely',\n   'high',\n   'Positive',\n   'Predictive',\n   'Value,',\n   'even',\n   'at',\n   'very',\n   'low',\n   'abundances',\n   'of',\n   'the',\n   'target',\n   'organism',\n   'in',\n   'a',\n   'mixed',\n   'sample.',\n   'SIANN',\n   'is',\n   'available',\n   'as',\n   'an',\n   'Illumina',\n   'BaseSpace',\n   'app,',\n   'as',\n   'well',\n   'as',\n   'through',\n   'Signature',\n   'Science,',\n   'LLC.',\n   'SIANN',\n   'results',\n   'are',\n   'presented',\n   'in',\n   'a',\n   'streamlined',\n   'report',\n   'designed',\n   'to',\n   'be',\n   'comprehensible',\n   'to',\n   'the',\n   'non-specialist',\n   'user,',\n   'providing',\n   'a',\n   'powerful',\n   'tool',\n   'for',\n   'rapid',\n   'species',\n   'detection',\n   'in',\n   'a',\n   'mixed',\n   'sample.',\n   'By',\n   'focusing',\n   'on',\n   'a',\n   'set',\n   'of',\n   '(customizable)',\n   'target',\n   'organisms',\n   'and',\n   'their',\n   'near',\n   'neighbors,',\n   'SIANN',\n   'can',\n   'operate',\n   'quickly',\n   'and',\n   'with',\n   'low',\n   'computational',\n   'requirements',\n   'while',\n   'delivering',\n   'highly',\n   'accurate',\n   'results.']),\n ('i9tbix2v',\n  ['An',\n   'emerging',\n   'disease',\n   'is',\n   'one',\n   'infectious',\n   'epidemic',\n   'caused',\n   'by',\n   'a',\n   'newly',\n   'transmissible',\n   'pathogen,',\n   'which',\n   'has',\n   'either',\n   'appeared',\n   'for',\n   'the',\n   'first',\n   'time',\n   'or',\n   'already',\n   'existed',\n   'in',\n   'human',\n   'populations,',\n   'having',\n   'the',\n   'capacity',\n   'to',\n   'increase',\n   'rapidly',\n   'in',\n   'incidence',\n   'as',\n   'well',\n   'as',\n   'geographic',\n   'range.',\n   'Adapting',\n   'to',\n   'human',\n   'immune',\n   'system,',\n   'emerging',\n   'diseases',\n   'may',\n   'trigger',\n   'large-scale',\n   'pandemic',\n   'spreading,',\n   'such',\n   'as',\n   'the',\n   'transnational',\n   'spreading',\n   'of',\n   'SARS,',\n   'the',\n   'global',\n   'outbreak',\n   'of',\n   'A(H1N1),',\n   'and',\n   'the',\n   'recent',\n   'potential',\n   'invasion',\n   'of',\n   'avian',\n   'influenza',\n   'A(H7N9).',\n   'To',\n   'study',\n   'the',\n   'dynamics',\n   'mediating',\n   'the',\n   'transmission',\n   'of',\n   'emerging',\n   'diseases,',\n   'spatial',\n   'epidemiology',\n   'of',\n   'networked',\n   'metapopulation',\n   'provides',\n   'a',\n   'valuable',\n   'modeling',\n   'framework,',\n   'which',\n   'takes',\n   'spatially',\n   'distributed',\n   'factors',\n   'into',\n   'consideration.',\n   'This',\n   'review',\n   'elaborates',\n   'the',\n   'latest',\n   'progresses',\n   'on',\n   'the',\n   'spatial',\n   'metapopulation',\n   'dynamics,',\n   'discusses',\n   'empirical',\n   'and',\n   'theoretical',\n   'findings',\n   'that',\n   'verify',\n   'the',\n   'validity',\n   'of',\n   'networked',\n   'metapopulations,',\n   'and',\n   'the',\n   'application',\n   'in',\n   'evaluating',\n   'the',\n   'effectiveness',\n   'of',\n   'disease',\n   'intervention',\n   'strategies',\n   'as',\n   'well.']),\n ('62gfisc6',\n  ['Germline',\n   'variation',\n   'at',\n   'immunoglobulin',\n   'gene',\n   '(IG)',\n   'loci',\n   'is',\n   'critical',\n   'for',\n   'pathogen-mediated',\n   'immunity,',\n   'but',\n   'establishing',\n   'complete',\n   'reference',\n   'sequences',\n   'in',\n   'these',\n   'regions',\n   'is',\n   'problematic',\n   'because',\n   'of',\n   'segmental',\n   'duplications',\n   'and',\n   'somatically',\n   'rearranged',\n   'source',\n   'DNA.',\n   'We',\n   'sequenced',\n   'BAC',\n   'clones',\n   'from',\n   'the',\n   'essentially',\n   'haploid',\n   'hydatidiform',\n   'mole,',\n   'CHM1,',\n   'across',\n   'the',\n   'light',\n   'chain',\n   'IG',\n   'loci,',\n   'kappa',\n   '(IGK)',\n   'and',\n   'lambda',\n   '(IGL),',\n   'creating',\n   'single',\n   'haplotype',\n   'representations',\n   'of',\n   'these',\n   'regions.',\n   'The',\n   'IGL',\n   'haplotype',\n   'is',\n   '1.25Mb',\n   'of',\n   'contiguous',\n   'sequence',\n   'with',\n   'four',\n   'novel',\n   'V',\n   'gene',\n   'and',\n   'one',\n   'novel',\n   'C',\n   'gene',\n   'alleles',\n   'and',\n   'an',\n   '11.9kbp',\n   'insertion.',\n   'The',\n   'IGK',\n   'haplotype',\n   'consists',\n   'of',\n   'two',\n   '644kbp',\n   'proximal',\n   'and',\n   '466kbp',\n   'distal',\n   'contigs',\n   'separated',\n   'by',\n   'a',\n   'gap',\n   'also',\n   'present',\n   'in',\n   'the',\n   'reference',\n   'genome',\n   'sequence.',\n   'Our',\n   'effort',\n   'added',\n   'an',\n   'additional',\n   '49kbp',\n   'of',\n   'unique',\n   'sequence',\n   'extending',\n   'into',\n   'this',\n   'gap.',\n   'The',\n   'IGK',\n   'haplotype',\n   'contains',\n   'six',\n   'novel',\n   'V',\n   'gene',\n   'and',\n   'one',\n   'novel',\n   'J',\n   'gene',\n   'alleles',\n   'and',\n   'a',\n   '16.7kbp',\n   'region',\n   'with',\n   'increased',\n   'sequence',\n   'identity',\n   'between',\n   'the',\n   'two',\n   'IGK',\n   'contigs,',\n   'exhibiting',\n   'signatures',\n   'of',\n   'interlocus',\n   'gene',\n   'conversion.',\n   'Our',\n   'data',\n   'facilitated',\n   'the',\n   'first',\n   'comparison',\n   'of',\n   'nucleotide',\n   'diversity',\n   'between',\n   'the',\n   'light',\n   'and',\n   'IG',\n   'heavy',\n   '(IGH)',\n   'chain',\n   'haplotypes',\n   'within',\n   'a',\n   'single',\n   'genome,',\n   'revealing',\n   'a',\n   'three',\n   'to',\n   'six',\n   'fold',\n   'enrichment',\n   'in',\n   'the',\n   'IGH',\n   'locus,',\n   'supporting',\n   'the',\n   'theory',\n   'that',\n   'the',\n   'heavy',\n   'chain',\n   'may',\n   'be',\n   'more',\n   'important',\n   'in',\n   'determining',\n   'antigenic',\n   'specificity.']),\n ('058r9486',\n  ['Deep',\n   'sequencing',\n   'of',\n   'clinical',\n   'samples',\n   'is',\n   'now',\n   'an',\n   'established',\n   'tool',\n   'for',\n   'the',\n   'detection',\n   'of',\n   'infectious',\n   'pathogens,',\n   'with',\n   'direct',\n   'medical',\n   'applications.',\n   'The',\n   'large',\n   'amount',\n   'of',\n   'data',\n   'generated',\n   'provides',\n   'an',\n   'opportunity',\n   'to',\n   'detect',\n   'species',\n   'even',\n   'at',\n   'very',\n   'low',\n   'levels,',\n   'provided',\n   'that',\n   'computational',\n   'tools',\n   'can',\n   'effectively',\n   'interpret',\n   'potentially',\n   'complex',\n   'metagenomic',\n   'mixtures.',\n   'Data',\n   'interpretation',\n   'is',\n   'complicated',\n   'by',\n   'the',\n   'fact',\n   'that',\n   'short',\n   'sequencing',\n   'reads',\n   'can',\n   'match',\n   'multiple',\n   'organisms',\n   'and',\n   'by',\n   'the',\n   'lack',\n   'of',\n   'completeness',\n   'of',\n   'existing',\n   'databases,',\n   'in',\n   'particular',\n   'for',\n   'viral',\n   'pathogens.',\n   'This',\n   'interpretation',\n   'problem',\n   'can',\n   'be',\n   'formulated',\n   'statistically',\n   'as',\n   'a',\n   'mixture',\n   'model,',\n   'where',\n   'the',\n   'species',\n   'of',\n   'origin',\n   'of',\n   'each',\n   'read',\n   'is',\n   'missing,',\n   'but',\n   'the',\n   'complete',\n   'knowledge',\n   'of',\n   'all',\n   'species',\n   'present',\n   'in',\n   'the',\n   'mixture',\n   'helps',\n   'with',\n   'the',\n   'individual',\n   'reads',\n   'assignment.',\n   'Several',\n   'analytical',\n   'tools',\n   'have',\n   'been',\n   'proposed',\n   'to',\n   'approximately',\n   'solve',\n   'this',\n   'computational',\n   'problem.',\n   'Here,',\n   'we',\n   'show',\n   'that',\n   'the',\n   'use',\n   'of',\n   'parallel',\n   'Monte',\n   'Carlo',\n   'Markov',\n   'chains',\n   '(MCMC)',\n   'for',\n   'the',\n   'exploration',\n   'of',\n   'the',\n   'species',\n   'space',\n   'enables',\n   'the',\n   'identification',\n   'of',\n   'the',\n   'set',\n   'of',\n   'species',\n   'most',\n   'likely',\n   'to',\n   'contribute',\n   'to',\n   'the',\n   'mixture.',\n   'The',\n   'added',\n   'accuracy',\n   'comes',\n   'at',\n   'a',\n   'cost',\n   'of',\n   'increased',\n   'computation',\n   'time.',\n   'Our',\n   'approach',\n   'is',\n   'useful',\n   'for',\n   'solving',\n   'complex',\n   'mixtures',\n   'involving',\n   'several',\n   'related',\n   'species.',\n   'We',\n   'designed',\n   'our',\n   'method',\n   'specifically',\n   'for',\n   'the',\n   'analysis',\n   'of',\n   'deep',\n   'transcriptome',\n   'sequencing',\n   'datasets',\n   'and',\n   'with',\n   'a',\n   'particular',\n   'focus',\n   'on',\n   'viral',\n   'pathogen',\n   'detection,',\n   'but',\n   'the',\n   'principles',\n   'are',\n   'applicable',\n   'more',\n   'generally',\n   'to',\n   'all',\n   'types',\n   'of',\n   'metagenomics',\n   'mixtures.',\n   'The',\n   'work',\n   'is',\n   'implemented',\n   'as',\n   'a',\n   'user',\n   'friendly',\n   'R',\n   'package,',\n   'available',\n   'from',\n   'CRAN:',\n   'http://cran.r-project.org/web/packages/metaMix']),\n ('wich35l7',\n  ['Developing',\n   'methods',\n   'to',\n   'reconstruct',\n   'transmission',\n   'histories',\n   'for',\n   'viral',\n   'outbreaks',\n   'could',\n   'provide',\n   'critical',\n   'information',\n   'to',\n   'support',\n   'locating',\n   'sources',\n   'of',\n   'disease',\n   'transmission.',\n   'Phylogenetic',\n   'methods',\n   'used',\n   'to',\n   'measure',\n   'the',\n   'degree',\n   'of',\n   'relatedness',\n   'among',\n   'sequenced',\n   'viral',\n   'samples',\n   'have',\n   'proven',\n   'useful',\n   'in',\n   'identifying',\n   'potential',\n   'outbreak',\n   'sources.',\n   'The',\n   'complex',\n   'nature',\n   'of',\n   'infectious',\n   'disease,',\n   'however,',\n   'makes',\n   'it',\n   'difficult',\n   'to',\n   'assign',\n   'a',\n   'rigorously',\n   'defined',\n   'quantitative',\n   'confidence',\n   'value',\n   'assessing',\n   'the',\n   'likelihood',\n   'of',\n   'a',\n   'true',\n   'direct',\n   'transmission',\n   'event',\n   'using',\n   'genetic',\n   'data',\n   'alone.',\n   'A',\n   'new',\n   'method',\n   'is',\n   'presented',\n   'to',\n   'calculate',\n   'a',\n   'confidence',\n   'value',\n   'assessing',\n   'the',\n   'likelihood',\n   'of',\n   'a',\n   'transmission',\n   'event',\n   'using',\n   'both',\n   'phylogenetic',\n   'inference',\n   'and',\n   'limited',\n   'knowledge',\n   'of',\n   'incubation',\n   'and',\n   'infectious',\n   'duration',\n   'times.',\n   'The',\n   'method',\n   'is',\n   'applied',\n   'to',\n   'simulations',\n   'of',\n   'a',\n   'foot',\n   'and',\n   'mouth',\n   'disease',\n   '(FMD)',\n   'outbreak',\n   'to',\n   'demonstrate',\n   'how',\n   'the',\n   'combination',\n   'of',\n   'both',\n   'phylogenetic',\n   'and',\n   'epidemiology',\n   'data',\n   'can',\n   'be',\n   'used',\n   'to',\n   'strengthen',\n   'the',\n   'assessment',\n   'of',\n   'the',\n   'likelihood',\n   'of',\n   'direct',\n   'transmission',\n   'over',\n   'methods',\n   'using',\n   'just',\n   'phylogenetic',\n   'data',\n   'or',\n   'infection',\n   'timing',\n   'data',\n   'alone.',\n   'The',\n   'method',\n   'is',\n   'applied',\n   'to',\n   'a',\n   'previous',\n   'FMD',\n   'outbreak',\n   'to',\n   'identify',\n   'areas',\n   'where',\n   'over',\n   'confidence',\n   'in',\n   'previously',\n   'inferred',\n   'direct',\n   'transmission',\n   'may',\n   'exist.',\n   'Combining',\n   'knowledge',\n   'from',\n   'viral',\n   'evolution',\n   'and',\n   'epidemiology',\n   'within',\n   'a',\n   'single',\n   'integrated',\n   'transmission',\n   'inference',\n   'framework',\n   'is',\n   'an',\n   'important',\n   'approach',\n   'to',\n   'assess',\n   'the',\n   'potential',\n   'likelihood',\n   'of',\n   'transmission',\n   'events',\n   'and',\n   'makes',\n   'clear',\n   'how',\n   'specific',\n   'features',\n   'of',\n   'a',\n   \"virus'\",\n   'spread',\n   'through',\n   'the',\n   'course',\n   'of',\n   'an',\n   'outbreak',\n   'will',\n   'directly',\n   'determine',\n   'the',\n   'potential',\n   'for',\n   'confidence',\n   'in',\n   'inferred',\n   'host',\n   'transmission',\n   'links.'])]","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[7]: [('vho70jcx',\n  ['Next-generation',\n   'sequencing',\n   'is',\n   'increasingly',\n   'being',\n   'used',\n   'to',\n   'study',\n   'samples',\n   'composed',\n   'of',\n   'mixtures',\n   'of',\n   'organisms,',\n   'such',\n   'as',\n   'in',\n   'clinical',\n   'applications',\n   'where',\n   'the',\n   'presence',\n   'of',\n   'a',\n   'pathogen',\n   'at',\n   'very',\n   'low',\n   'abundance',\n   'may',\n   'be',\n   'highly',\n   'important.',\n   'We',\n   'present',\n   'an',\n   'analytical',\n   'method',\n   '(SIANN:',\n   'Strain',\n   'Identification',\n   'by',\n   'Alignment',\n   'to',\n   'Near',\n   'Neighbors)',\n   'specifically',\n   'designed',\n   'to',\n   'rapidly',\n   'detect',\n   'a',\n   'set',\n   'of',\n   'target',\n   'organisms',\n   'in',\n   'mixed',\n   'samples',\n   'that',\n   'achieves',\n   'a',\n   'high',\n   'degree',\n   'of',\n   'species-',\n   'and',\n   'strain-specificity',\n   'by',\n   'aligning',\n   'short',\n   'sequence',\n   'reads',\n   'to',\n   'the',\n   'genomes',\n   'of',\n   'near',\n   'neighbor',\n   'organisms,',\n   'as',\n   'well',\n   'as',\n   'that',\n   'of',\n   'the',\n   'target.',\n   'Empirical',\n   'benchmarking',\n   'alongside',\n   'the',\n   'current',\n   'state-of-the-art',\n   'methods',\n   'shows',\n   'an',\n   'extremely',\n   'high',\n   'Positive',\n   'Predictive',\n   'Value,',\n   'even',\n   'at',\n   'very',\n   'low',\n   'abundances',\n   'of',\n   'the',\n   'target',\n   'organism',\n   'in',\n   'a',\n   'mixed',\n   'sample.',\n   'SIANN',\n   'is',\n   'available',\n   'as',\n   'an',\n   'Illumina',\n   'BaseSpace',\n   'app,',\n   'as',\n   'well',\n   'as',\n   'through',\n   'Signature',\n   'Science,',\n   'LLC.',\n   'SIANN',\n   'results',\n   'are',\n   'presented',\n   'in',\n   'a',\n   'streamlined',\n   'report',\n   'designed',\n   'to',\n   'be',\n   'comprehensible',\n   'to',\n   'the',\n   'non-specialist',\n   'user,',\n   'providing',\n   'a',\n   'powerful',\n   'tool',\n   'for',\n   'rapid',\n   'species',\n   'detection',\n   'in',\n   'a',\n   'mixed',\n   'sample.',\n   'By',\n   'focusing',\n   'on',\n   'a',\n   'set',\n   'of',\n   '(customizable)',\n   'target',\n   'organisms',\n   'and',\n   'their',\n   'near',\n   'neighbors,',\n   'SIANN',\n   'can',\n   'operate',\n   'quickly',\n   'and',\n   'with',\n   'low',\n   'computational',\n   'requirements',\n   'while',\n   'delivering',\n   'highly',\n   'accurate',\n   'results.']),\n ('i9tbix2v',\n  ['An',\n   'emerging',\n   'disease',\n   'is',\n   'one',\n   'infectious',\n   'epidemic',\n   'caused',\n   'by',\n   'a',\n   'newly',\n   'transmissible',\n   'pathogen,',\n   'which',\n   'has',\n   'either',\n   'appeared',\n   'for',\n   'the',\n   'first',\n   'time',\n   'or',\n   'already',\n   'existed',\n   'in',\n   'human',\n   'populations,',\n   'having',\n   'the',\n   'capacity',\n   'to',\n   'increase',\n   'rapidly',\n   'in',\n   'incidence',\n   'as',\n   'well',\n   'as',\n   'geographic',\n   'range.',\n   'Adapting',\n   'to',\n   'human',\n   'immune',\n   'system,',\n   'emerging',\n   'diseases',\n   'may',\n   'trigger',\n   'large-scale',\n   'pandemic',\n   'spreading,',\n   'such',\n   'as',\n   'the',\n   'transnational',\n   'spreading',\n   'of',\n   'SARS,',\n   'the',\n   'global',\n   'outbreak',\n   'of',\n   'A(H1N1),',\n   'and',\n   'the',\n   'recent',\n   'potential',\n   'invasion',\n   'of',\n   'avian',\n   'influenza',\n   'A(H7N9).',\n   'To',\n   'study',\n   'the',\n   'dynamics',\n   'mediating',\n   'the',\n   'transmission',\n   'of',\n   'emerging',\n   'diseases,',\n   'spatial',\n   'epidemiology',\n   'of',\n   'networked',\n   'metapopulation',\n   'provides',\n   'a',\n   'valuable',\n   'modeling',\n   'framework,',\n   'which',\n   'takes',\n   'spatially',\n   'distributed',\n   'factors',\n   'into',\n   'consideration.',\n   'This',\n   'review',\n   'elaborates',\n   'the',\n   'latest',\n   'progresses',\n   'on',\n   'the',\n   'spatial',\n   'metapopulation',\n   'dynamics,',\n   'discusses',\n   'empirical',\n   'and',\n   'theoretical',\n   'findings',\n   'that',\n   'verify',\n   'the',\n   'validity',\n   'of',\n   'networked',\n   'metapopulations,',\n   'and',\n   'the',\n   'application',\n   'in',\n   'evaluating',\n   'the',\n   'effectiveness',\n   'of',\n   'disease',\n   'intervention',\n   'strategies',\n   'as',\n   'well.']),\n ('62gfisc6',\n  ['Germline',\n   'variation',\n   'at',\n   'immunoglobulin',\n   'gene',\n   '(IG)',\n   'loci',\n   'is',\n   'critical',\n   'for',\n   'pathogen-mediated',\n   'immunity,',\n   'but',\n   'establishing',\n   'complete',\n   'reference',\n   'sequences',\n   'in',\n   'these',\n   'regions',\n   'is',\n   'problematic',\n   'because',\n   'of',\n   'segmental',\n   'duplications',\n   'and',\n   'somatically',\n   'rearranged',\n   'source',\n   'DNA.',\n   'We',\n   'sequenced',\n   'BAC',\n   'clones',\n   'from',\n   'the',\n   'essentially',\n   'haploid',\n   'hydatidiform',\n   'mole,',\n   'CHM1,',\n   'across',\n   'the',\n   'light',\n   'chain',\n   'IG',\n   'loci,',\n   'kappa',\n   '(IGK)',\n   'and',\n   'lambda',\n   '(IGL),',\n   'creating',\n   'single',\n   'haplotype',\n   'representations',\n   'of',\n   'these',\n   'regions.',\n   'The',\n   'IGL',\n   'haplotype',\n   'is',\n   '1.25Mb',\n   'of',\n   'contiguous',\n   'sequence',\n   'with',\n   'four',\n   'novel',\n   'V',\n   'gene',\n   'and',\n   'one',\n   'novel',\n   'C',\n   'gene',\n   'alleles',\n   'and',\n   'an',\n   '11.9kbp',\n   'insertion.',\n   'The',\n   'IGK',\n   'haplotype',\n   'consists',\n   'of',\n   'two',\n   '644kbp',\n   'proximal',\n   'and',\n   '466kbp',\n   'distal',\n   'contigs',\n   'separated',\n   'by',\n   'a',\n   'gap',\n   'also',\n   'present',\n   'in',\n   'the',\n   'reference',\n   'genome',\n   'sequence.',\n   'Our',\n   'effort',\n   'added',\n   'an',\n   'additional',\n   '49kbp',\n   'of',\n   'unique',\n   'sequence',\n   'extending',\n   'into',\n   'this',\n   'gap.',\n   'The',\n   'IGK',\n   'haplotype',\n   'contains',\n   'six',\n   'novel',\n   'V',\n   'gene',\n   'and',\n   'one',\n   'novel',\n   'J',\n   'gene',\n   'alleles',\n   'and',\n   'a',\n   '16.7kbp',\n   'region',\n   'with',\n   'increased',\n   'sequence',\n   'identity',\n   'between',\n   'the',\n   'two',\n   'IGK',\n   'contigs,',\n   'exhibiting',\n   'signatures',\n   'of',\n   'interlocus',\n   'gene',\n   'conversion.',\n   'Our',\n   'data',\n   'facilitated',\n   'the',\n   'first',\n   'comparison',\n   'of',\n   'nucleotide',\n   'diversity',\n   'between',\n   'the',\n   'light',\n   'and',\n   'IG',\n   'heavy',\n   '(IGH)',\n   'chain',\n   'haplotypes',\n   'within',\n   'a',\n   'single',\n   'genome,',\n   'revealing',\n   'a',\n   'three',\n   'to',\n   'six',\n   'fold',\n   'enrichment',\n   'in',\n   'the',\n   'IGH',\n   'locus,',\n   'supporting',\n   'the',\n   'theory',\n   'that',\n   'the',\n   'heavy',\n   'chain',\n   'may',\n   'be',\n   'more',\n   'important',\n   'in',\n   'determining',\n   'antigenic',\n   'specificity.']),\n ('058r9486',\n  ['Deep',\n   'sequencing',\n   'of',\n   'clinical',\n   'samples',\n   'is',\n   'now',\n   'an',\n   'established',\n   'tool',\n   'for',\n   'the',\n   'detection',\n   'of',\n   'infectious',\n   'pathogens,',\n   'with',\n   'direct',\n   'medical',\n   'applications.',\n   'The',\n   'large',\n   'amount',\n   'of',\n   'data',\n   'generated',\n   'provides',\n   'an',\n   'opportunity',\n   'to',\n   'detect',\n   'species',\n   'even',\n   'at',\n   'very',\n   'low',\n   'levels,',\n   'provided',\n   'that',\n   'computational',\n   'tools',\n   'can',\n   'effectively',\n   'interpret',\n   'potentially',\n   'complex',\n   'metagenomic',\n   'mixtures.',\n   'Data',\n   'interpretation',\n   'is',\n   'complicated',\n   'by',\n   'the',\n   'fact',\n   'that',\n   'short',\n   'sequencing',\n   'reads',\n   'can',\n   'match',\n   'multiple',\n   'organisms',\n   'and',\n   'by',\n   'the',\n   'lack',\n   'of',\n   'completeness',\n   'of',\n   'existing',\n   'databases,',\n   'in',\n   'particular',\n   'for',\n   'viral',\n   'pathogens.',\n   'This',\n   'interpretation',\n   'problem',\n   'can',\n   'be',\n   'formulated',\n   'statistically',\n   'as',\n   'a',\n   'mixture',\n   'model,',\n   'where',\n   'the',\n   'species',\n   'of',\n   'origin',\n   'of',\n   'each',\n   'read',\n   'is',\n   'missing,',\n   'but',\n   'the',\n   'complete',\n   'knowledge',\n   'of',\n   'all',\n   'species',\n   'present',\n   'in',\n   'the',\n   'mixture',\n   'helps',\n   'with',\n   'the',\n   'individual',\n   'reads',\n   'assignment.',\n   'Several',\n   'analytical',\n   'tools',\n   'have',\n   'been',\n   'proposed',\n   'to',\n   'approximately',\n   'solve',\n   'this',\n   'computational',\n   'problem.',\n   'Here,',\n   'we',\n   'show',\n   'that',\n   'the',\n   'use',\n   'of',\n   'parallel',\n   'Monte',\n   'Carlo',\n   'Markov',\n   'chains',\n   '(MCMC)',\n   'for',\n   'the',\n   'exploration',\n   'of',\n   'the',\n   'species',\n   'space',\n   'enables',\n   'the',\n   'identification',\n   'of',\n   'the',\n   'set',\n   'of',\n   'species',\n   'most',\n   'likely',\n   'to',\n   'contribute',\n   'to',\n   'the',\n   'mixture.',\n   'The',\n   'added',\n   'accuracy',\n   'comes',\n   'at',\n   'a',\n   'cost',\n   'of',\n   'increased',\n   'computation',\n   'time.',\n   'Our',\n   'approach',\n   'is',\n   'useful',\n   'for',\n   'solving',\n   'complex',\n   'mixtures',\n   'involving',\n   'several',\n   'related',\n   'species.',\n   'We',\n   'designed',\n   'our',\n   'method',\n   'specifically',\n   'for',\n   'the',\n   'analysis',\n   'of',\n   'deep',\n   'transcriptome',\n   'sequencing',\n   'datasets',\n   'and',\n   'with',\n   'a',\n   'particular',\n   'focus',\n   'on',\n   'viral',\n   'pathogen',\n   'detection,',\n   'but',\n   'the',\n   'principles',\n   'are',\n   'applicable',\n   'more',\n   'generally',\n   'to',\n   'all',\n   'types',\n   'of',\n   'metagenomics',\n   'mixtures.',\n   'The',\n   'work',\n   'is',\n   'implemented',\n   'as',\n   'a',\n   'user',\n   'friendly',\n   'R',\n   'package,',\n   'available',\n   'from',\n   'CRAN:',\n   'http://cran.r-project.org/web/packages/metaMix']),\n ('wich35l7',\n  ['Developing',\n   'methods',\n   'to',\n   'reconstruct',\n   'transmission',\n   'histories',\n   'for',\n   'viral',\n   'outbreaks',\n   'could',\n   'provide',\n   'critical',\n   'information',\n   'to',\n   'support',\n   'locating',\n   'sources',\n   'of',\n   'disease',\n   'transmission.',\n   'Phylogenetic',\n   'methods',\n   'used',\n   'to',\n   'measure',\n   'the',\n   'degree',\n   'of',\n   'relatedness',\n   'among',\n   'sequenced',\n   'viral',\n   'samples',\n   'have',\n   'proven',\n   'useful',\n   'in',\n   'identifying',\n   'potential',\n   'outbreak',\n   'sources.',\n   'The',\n   'complex',\n   'nature',\n   'of',\n   'infectious',\n   'disease,',\n   'however,',\n   'makes',\n   'it',\n   'difficult',\n   'to',\n   'assign',\n   'a',\n   'rigorously',\n   'defined',\n   'quantitative',\n   'confidence',\n   'value',\n   'assessing',\n   'the',\n   'likelihood',\n   'of',\n   'a',\n   'true',\n   'direct',\n   'transmission',\n   'event',\n   'using',\n   'genetic',\n   'data',\n   'alone.',\n   'A',\n   'new',\n   'method',\n   'is',\n   'presented',\n   'to',\n   'calculate',\n   'a',\n   'confidence',\n   'value',\n   'assessing',\n   'the',\n   'likelihood',\n   'of',\n   'a',\n   'transmission',\n   'event',\n   'using',\n   'both',\n   'phylogenetic',\n   'inference',\n   'and',\n   'limited',\n   'knowledge',\n   'of',\n   'incubation',\n   'and',\n   'infectious',\n   'duration',\n   'times.',\n   'The',\n   'method',\n   'is',\n   'applied',\n   'to',\n   'simulations',\n   'of',\n   'a',\n   'foot',\n   'and',\n   'mouth',\n   'disease',\n   '(FMD)',\n   'outbreak',\n   'to',\n   'demonstrate',\n   'how',\n   'the',\n   'combination',\n   'of',\n   'both',\n   'phylogenetic',\n   'and',\n   'epidemiology',\n   'data',\n   'can',\n   'be',\n   'used',\n   'to',\n   'strengthen',\n   'the',\n   'assessment',\n   'of',\n   'the',\n   'likelihood',\n   'of',\n   'direct',\n   'transmission',\n   'over',\n   'methods',\n   'using',\n   'just',\n   'phylogenetic',\n   'data',\n   'or',\n   'infection',\n   'timing',\n   'data',\n   'alone.',\n   'The',\n   'method',\n   'is',\n   'applied',\n   'to',\n   'a',\n   'previous',\n   'FMD',\n   'outbreak',\n   'to',\n   'identify',\n   'areas',\n   'where',\n   'over',\n   'confidence',\n   'in',\n   'previously',\n   'inferred',\n   'direct',\n   'transmission',\n   'may',\n   'exist.',\n   'Combining',\n   'knowledge',\n   'from',\n   'viral',\n   'evolution',\n   'and',\n   'epidemiology',\n   'within',\n   'a',\n   'single',\n   'integrated',\n   'transmission',\n   'inference',\n   'framework',\n   'is',\n   'an',\n   'important',\n   'approach',\n   'to',\n   'assess',\n   'the',\n   'potential',\n   'likelihood',\n   'of',\n   'transmission',\n   'events',\n   'and',\n   'makes',\n   'clear',\n   'how',\n   'specific',\n   'features',\n   'of',\n   'a',\n   \"virus'\",\n   'spread',\n   'through',\n   'the',\n   'course',\n   'of',\n   'an',\n   'outbreak',\n   'will',\n   'directly',\n   'determine',\n   'the',\n   'potential',\n   'for',\n   'confidence',\n   'in',\n   'inferred',\n   'host',\n   'transmission',\n   'links.'])]"]}}],"execution_count":0},{"cell_type":"code","source":["dbutils.fs.ls('/FileStore/tables')"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"e0cb7cc2-83d2-4746-b979-265d43297efa"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[8]: [FileInfo(path='dbfs:/FileStore/tables/accounts.zip', name='accounts.zip', size=5297592, modificationTime=1645020773000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial/', name='clinicaltrial/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2019_csv', name='clinicaltrial_2019_csv', size=42400056, modificationTime=1648652752000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2019_csv.gz', name='clinicaltrial_2019_csv.gz', size=10060669, modificationTime=1648639910000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2020_csv.gz', name='clinicaltrial_2020_csv.gz', size=10981608, modificationTime=1648639888000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2021.csv', name='clinicaltrial_2021.csv', size=50359696, modificationTime=1648898522000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2021_csv.gz', name='clinicaltrial_2021_csv.gz', size=11921810, modificationTime=1648639873000),\n FileInfo(path='dbfs:/FileStore/tables/conditions/', name='conditions/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/conditions_tree/', name='conditions_tree/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/devicestatus.zip', name='devicestatus.zip', size=23873574, modificationTime=1645213614000),\n FileInfo(path='dbfs:/FileStore/tables/json_activations.zip', name='json_activations.zip', size=8411369, modificationTime=1645009487000),\n FileInfo(path='dbfs:/FileStore/tables/letter_frequencies.txt', name='letter_frequencies.txt', size=2593894, modificationTime=1646109824000),\n FileInfo(path='dbfs:/FileStore/tables/logs.zip', name='logs.zip', size=18168065, modificationTime=1645021105000),\n FileInfo(path='dbfs:/FileStore/tables/mesh/', name='mesh/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/mesh.csv', name='mesh.csv', size=5295548, modificationTime=1651014985000),\n FileInfo(path='dbfs:/FileStore/tables/negative_words.csv', name='negative_words.csv', size=49541, modificationTime=1651070229000),\n FileInfo(path='dbfs:/FileStore/tables/pharma.csv', name='pharma.csv', size=678999, modificationTime=1648639822000),\n FileInfo(path='dbfs:/FileStore/tables/positive_words.csv', name='positive_words.csv', size=21097, modificationTime=1651070229000),\n FileInfo(path='dbfs:/FileStore/tables/webpage.zip', name='webpage.zip', size=1582, modificationTime=1646219627000)]","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[8]: [FileInfo(path='dbfs:/FileStore/tables/accounts.zip', name='accounts.zip', size=5297592, modificationTime=1645020773000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial/', name='clinicaltrial/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2019_csv', name='clinicaltrial_2019_csv', size=42400056, modificationTime=1648652752000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2019_csv.gz', name='clinicaltrial_2019_csv.gz', size=10060669, modificationTime=1648639910000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2020_csv.gz', name='clinicaltrial_2020_csv.gz', size=10981608, modificationTime=1648639888000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2021.csv', name='clinicaltrial_2021.csv', size=50359696, modificationTime=1648898522000),\n FileInfo(path='dbfs:/FileStore/tables/clinicaltrial_2021_csv.gz', name='clinicaltrial_2021_csv.gz', size=11921810, modificationTime=1648639873000),\n FileInfo(path='dbfs:/FileStore/tables/conditions/', name='conditions/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/conditions_tree/', name='conditions_tree/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/devicestatus.zip', name='devicestatus.zip', size=23873574, modificationTime=1645213614000),\n FileInfo(path='dbfs:/FileStore/tables/json_activations.zip', name='json_activations.zip', size=8411369, modificationTime=1645009487000),\n FileInfo(path='dbfs:/FileStore/tables/letter_frequencies.txt', name='letter_frequencies.txt', size=2593894, modificationTime=1646109824000),\n FileInfo(path='dbfs:/FileStore/tables/logs.zip', name='logs.zip', size=18168065, modificationTime=1645021105000),\n FileInfo(path='dbfs:/FileStore/tables/mesh/', name='mesh/', size=0, modificationTime=0),\n FileInfo(path='dbfs:/FileStore/tables/mesh.csv', name='mesh.csv', size=5295548, modificationTime=1651014985000),\n FileInfo(path='dbfs:/FileStore/tables/negative_words.csv', name='negative_words.csv', size=49541, modificationTime=1651070229000),\n FileInfo(path='dbfs:/FileStore/tables/pharma.csv', name='pharma.csv', size=678999, modificationTime=1648639822000),\n FileInfo(path='dbfs:/FileStore/tables/positive_words.csv', name='positive_words.csv', size=21097, modificationTime=1651070229000),\n FileInfo(path='dbfs:/FileStore/tables/webpage.zip', name='webpage.zip', size=1582, modificationTime=1646219627000)]"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 5"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"41531c81-0fab-4976-84d6-cece6227198e"}}},{"cell_type":"code","source":["positiveWordsDF = spark.read.csv('dbfs:/FileStore/tables/positive_words.csv')\npositiveWordsDF.show(truncate=False)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"f73ef2e0-66b1-42b8-bb09-b591d614f750"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"+---------------+\n|_c0            |\n+---------------+\n|a+             |\n|abound         |\n|abounds        |\n|abundance      |\n|abundant       |\n|accessable     |\n|accessible     |\n|acclaim        |\n|acclaimed      |\n|acclamation    |\n|accolade       |\n|accolades      |\n|accommodative  |\n|accomodative   |\n|accomplish     |\n|accomplished   |\n|accomplishment |\n|accomplishments|\n|accurate       |\n|accurately     |\n+---------------+\nonly showing top 20 rows\n\n","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["+---------------+\n|_c0            |\n+---------------+\n|a+             |\n|abound         |\n|abounds        |\n|abundance      |\n|abundant       |\n|accessable     |\n|accessible     |\n|acclaim        |\n|acclaimed      |\n|acclamation    |\n|accolade       |\n|accolades      |\n|accommodative  |\n|accomodative   |\n|accomplish     |\n|accomplished   |\n|accomplishment |\n|accomplishments|\n|accurate       |\n|accurately     |\n+---------------+\nonly showing top 20 rows\n\n"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 6"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"1bacaad2-39ed-4d1c-bf2c-3ffca72348b9"}}},{"cell_type":"code","source":["negativeWordsDF = spark.read.csv('dbfs:/FileStore/tables/negative_words.csv')\nnegativeWordsDF.show(truncate=False)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"95adba84-ebf1-4c6a-9883-4c14cf9ef57f"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"+-------------+\n|_c0          |\n+-------------+\n|2-faced      |\n|2-faces      |\n|abnormal     |\n|abolish      |\n|abominable   |\n|abominably   |\n|abominate    |\n|abomination  |\n|abort        |\n|aborted      |\n|aborts       |\n|abrade       |\n|abrasive     |\n|abrupt       |\n|abruptly     |\n|abscond      |\n|absence      |\n|absent-minded|\n|absentee     |\n|absurd       |\n+-------------+\nonly showing top 20 rows\n\n","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["+-------------+\n|_c0          |\n+-------------+\n|2-faced      |\n|2-faces      |\n|abnormal     |\n|abolish      |\n|abominable   |\n|abominably   |\n|abominate    |\n|abomination  |\n|abort        |\n|aborted      |\n|aborts       |\n|abrade       |\n|abrasive     |\n|abrupt       |\n|abruptly     |\n|abscond      |\n|absence      |\n|absent-minded|\n|absentee     |\n|absurd       |\n+-------------+\nonly showing top 20 rows\n\n"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 7"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"69ff9742-0580-47fb-b2bb-c6f298ef482c"}}},{"cell_type":"code","source":["positive_words_set = set(positiveWordsDF.select('_c0').rdd.flatMap(lambda x : x).collect())\npositive_words_set"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"b3cf6d55-352b-4706-9d93-3797fc9d1996"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[11]: {'uncomplicated',\n 'pain-free',\n 'impressed',\n 'afford',\n 'afordable',\n 'gained',\n 'positively',\n 'justly',\n 'courageously',\n 'enthralled',\n 'genuine',\n 'satisfy',\n 'fertile',\n 'sagacity',\n 'beckon',\n 'firmer',\n 'luster',\n 'invaluablely',\n 'right',\n 'endearing',\n 'fame',\n 'plusses',\n 'titillatingly',\n 'vivid',\n 'faithful',\n 'gush',\n 'improvements',\n 'consistently',\n 'beneficial',\n 'ideal',\n 'inspiring',\n 'enjoyable',\n 'endorsed',\n 'futurestic',\n 'charmingly',\n 'rejoicingly',\n 'sufficed',\n 'charisma',\n 'wisely',\n 'favored',\n 'equitable',\n 'ingenious',\n 'dead-cheap',\n 'spellbinding',\n 'coherent',\n 'exhilarate',\n 'grin',\n 'prosper',\n 'benifits',\n 'gorgeously',\n 'successes',\n 'commodious',\n 'fervid',\n 'adventuresome',\n 'excited',\n 'intelligible',\n 'irresistibly',\n 'dignify',\n 'hardier',\n 'openly',\n 'politeness',\n 'reforming',\n 'preeminent',\n 'supporting',\n 'ample',\n 'honesty',\n 'leads',\n 'illustrious',\n 'obtainable',\n 'generously',\n 'proud',\n 'piety',\n 'satisfied',\n 'painlessly',\n 'gloriously',\n 'covenant',\n 'angelic',\n 'prodigy',\n 'abounds',\n 'impartiality',\n 'overtaking',\n 'decency',\n 'stirringly',\n 'diligence',\n 'enthuse',\n 'quicker',\n 'delight',\n 'chivalry',\n 'awesomeness',\n 'jovial',\n 'whoooa',\n 'immense',\n 'available',\n 'honest',\n 'winnable',\n 'prestigious',\n 'acclaim',\n 'mastery',\n 'futuristic',\n 'properly',\n 'better-known',\n 'dominated',\n 'adroitly',\n 'shimmeringly',\n 'fantastic',\n 'unwavering',\n 'feasible',\n 'sagely',\n 'beckons',\n 'refund',\n 'high-quality',\n 'hilarious',\n 'chivalrous',\n 'gratitude',\n 'prominent',\n 'remedy',\n 'inspirational',\n 'awestruck',\n 'fave',\n 'enterprising',\n 'warmth',\n 'well-behaved',\n 'affirm',\n 'swankiest',\n 'revival',\n 'low-price',\n 'earnest',\n 'thrift',\n 'appeal',\n 'fanfare',\n 'felicitate',\n 'bloom',\n 'favorited',\n 'rightly',\n 'dexterously',\n 'sturdier',\n 'user-friendly',\n 'beckoned',\n 'ovation',\n 'gaiety',\n 'valiantly',\n 'freed',\n 'remarkably',\n 'strikingly',\n 'stupendous',\n 'surpass',\n 'easygoing',\n 'truthful',\n 'well-wishers',\n 'humble',\n 'foolproof',\n 'rational',\n 'hottest',\n 'streamlined',\n 'brisk',\n 'luckiest',\n 'fav',\n 'enchantingly',\n 'elevate',\n 'destiny',\n 'exalted',\n 'subsidizes',\n 'polished',\n 'consistent',\n 'realistic',\n 'hopeful',\n 'classic',\n 'cuteness',\n 'overtook',\n 'truthfully',\n 'peaceable',\n 'sensibly',\n 'substantive',\n 'steadiness',\n 'insightfully',\n 'glitter',\n 'appreciated',\n 'magical',\n 'razor-sharp',\n 'faith',\n 'congratulatory',\n 'energy-saving',\n 'reasonably',\n 'vigilant',\n 'pleasantly',\n 'stimulates',\n 'fine-looking',\n 'fast-paced',\n 'frugal',\n 'righteous',\n 'benevolent',\n 'unquestionably',\n 'meritorious',\n 'prosperous',\n 'well-balanced',\n 'fidelity',\n 'leading',\n 'vibrantly',\n 'dazzle',\n 'affability',\n 'pride',\n 'exultant',\n 'geeky',\n 'halcyon',\n 'tranquility',\n 'outperform',\n 'splendor',\n 'freedoms',\n 'liking',\n 'virtue',\n 'soundness',\n 'smiles',\n 'supreme',\n 'worked',\n 'cohere',\n 'reformed',\n 'important',\n 'thrillingly',\n 'stunned',\n 'approve',\n 'promise',\n 'amazing',\n 'cleanest',\n 'flashy',\n 'superiority',\n 'dead-on',\n 'remission',\n 'evocative',\n 'seamless',\n 'accessible',\n 'standout',\n 'sustainability',\n 'effusive',\n 'festive',\n 'delightfulness',\n 'gainful',\n 'unforgettable',\n 'reassurance',\n 'thrive',\n 'amiabily',\n 'outwit',\n 'relent',\n 'gleeful',\n 'champ',\n 'venerate',\n 'savior',\n 'advantage',\n 'raptureous',\n 'enjoyed',\n 'breakthrough',\n 'long-lasting',\n 'simplify',\n 'top',\n 'appreciable',\n 'handsome',\n 'swift',\n 'capable',\n 'proper',\n 'gallant',\n 'inviolate',\n 'adored',\n 'supple',\n 'promoter',\n 'sharpest',\n 'superbly',\n 'dependably',\n 'accomplishments',\n 'merciful',\n 'gracefully',\n 'convience',\n 'enthusiastically',\n 'examplary',\n 'revere',\n 'hardy',\n 'happier',\n 'fine',\n 'soothe',\n 'lean',\n 'indulgence',\n 'excitingly',\n 'eyecatching',\n 'thrilled',\n 'appreciative',\n 'awesomely',\n 'magnificence',\n 'majestic',\n 'elation',\n 'variety',\n 'inviolable',\n 'smoother',\n 'cooperative',\n 'sophisticated',\n 'shiny',\n 'hallmarks',\n 'adoringly',\n 'intriguing',\n 'skilled',\n 'refinement',\n 'loveliness',\n 'recommendations',\n 'ecstatically',\n 'mind-blowing',\n 'inestimable',\n 'achievement',\n 'viewable',\n 'evenly',\n 'sublime',\n 'liberty',\n 'danke',\n 'top-notch',\n 'contrasty',\n 'renowned',\n 'gladden',\n 'faultless',\n 'unrivaled',\n 'prudence',\n 'inspiration',\n 'winner',\n 'skillfully',\n 'admirably',\n 'dashing',\n 'dexterous',\n 'god-send',\n 'perfection',\n 'win',\n 'clever',\n 'wholeheartedly',\n 'slammin',\n 'spirited',\n 'peerless',\n 'eloquence',\n 'amazes',\n 'effective',\n 'boom',\n 'gratifies',\n 'enhances',\n 'credence',\n 'earnestness',\n 'striking',\n 'deserving',\n 'friendliness',\n 'motivated',\n 'gusto',\n 'helpful',\n 'harmless',\n 'gladness',\n 'bolster',\n 'idyllic',\n 'encourage',\n 'heal',\n 'ilu',\n 'jubilantly',\n 'brave',\n 'readable',\n 'healthy',\n 'progressive',\n 'regard',\n 'wowing',\n 'innovative',\n 'wowed',\n 'good',\n 'facilitate',\n 'entrust',\n 'immaculate',\n 'unlimited',\n 'nicest',\n 'impartial',\n 'improvement',\n 'fervent',\n 'simplifies',\n 'impeccable',\n 'hands-down',\n 'useable',\n 'exceeded',\n 'fans',\n 'pampered',\n 'tidy',\n 'satisified',\n 'calming',\n 'successful',\n 'attractively',\n 'innocuous',\n 'bountiful',\n 'rosy',\n 'multi-purpose',\n 'smoothly',\n 'savings',\n 'credible',\n 'resplendent',\n 'carefree',\n 'excelled',\n 'master',\n 'euphoric',\n 'generous',\n 'perfect',\n 'irresistible',\n 'assurance',\n 'stimulate',\n 'gem',\n 'passionate',\n 'humane',\n 'competitive',\n 'suave',\n 'well-connected',\n 'world-famous',\n 'enjoyment',\n 'glisten',\n 'self-satisfaction',\n 'sweetness',\n 'sensational',\n 'civilize',\n 'complement',\n 'fascinate',\n 'enrapt',\n 'accolades',\n 'exaltingly',\n 'gains',\n 'windfall',\n 'industrious',\n 'well-made',\n 'respectful',\n 'understandable',\n 'eagerly',\n 'eyecatch',\n 'fortitude',\n 'revolutionary',\n 'splendidly',\n 'efficacious',\n 'astonishingly',\n 'willing',\n 'famed',\n 'inpressed',\n 'user-replaceable',\n 'luxuriously',\n 'exquisite',\n 'righten',\n 'outperforms',\n 'amicably',\n 'righteousness',\n 'stellar',\n 'skillful',\n 'coolest',\n 'adulatory',\n 'suffices',\n 'prowess',\n 'fearless',\n 'decisive',\n 'positives',\n 'flutter',\n 'enliven',\n 'beneficent',\n 'exuberance',\n 'topnotch',\n 'adaptable',\n 'maturely',\n 'well-known',\n 'ecstasies',\n 'charitable',\n 'renown',\n 'commendably',\n 'favour',\n 'restructuring',\n 'modest',\n 'renaissance',\n 'amiable',\n 'courage',\n 'tenderly',\n 'reverent',\n 'magic',\n 'plentiful',\n 'shine',\n 'pepping',\n 'insightful',\n 'well-positioned',\n 'richness',\n 'thoughtfully',\n 'excel',\n 'conciliatory',\n 'tingle',\n 'fervidly',\n 'astounding',\n 'prodigious',\n 'issue-free',\n 'lawful',\n 'beautifully',\n 'restructure',\n 'lionhearted',\n 'jollify',\n 'sensitive',\n 'devout',\n 'cherub',\n 'attentive',\n 'honoring',\n 'trumpet',\n 'warmhearted',\n 'simplest',\n 'suffice',\n 'ambitious',\n 'cost-effective',\n 'sweetly',\n 'self-determination',\n 'overture',\n 'wise',\n 'refined',\n 'undaunted',\n 'work',\n 'wondrous',\n 'mature',\n 'balanced',\n 'steadfast',\n 'solidarity',\n 'pretty',\n 'cleared',\n 'contribution',\n 'neatest',\n 'profusion',\n 'non-violence',\n 'radiance',\n 'lovely',\n 'exhilaration',\n 'amenity',\n 'gaily',\n 'smitten',\n 'wows',\n 'rightness',\n 'commendable',\n 'undisputed',\n 'pleasurably',\n 'easiness',\n 'cool',\n 'conveniently',\n 'talented',\n 'bonuses',\n 'convient',\n 'effusiveness',\n 'exceeds',\n 'commend',\n 'warmly',\n 'poignant',\n 'glistening',\n 'reward',\n 'works',\n 'stabilize',\n 'elegant',\n 'lighter',\n 'prodigiously',\n 'stimulative',\n 'favor',\n 'galore',\n 'ennoble',\n 'wonderous',\n 'handsomely',\n 'super',\n 'enviable',\n 'greatest',\n 'admiringly',\n 'self-sufficient',\n 'astoundingly',\n 'leverage',\n 'outperformed',\n 'appreciatively',\n 'legendary',\n 'congenial',\n 'panoramic',\n 'exalting',\n 'thank',\n 'affection',\n 'fascinatingly',\n 'pure',\n 'avid',\n 'twinkly',\n 'fortune',\n 'clearer',\n 'decisiveness',\n 'sharper',\n 'unreal',\n 'undamaged',\n 'ebullience',\n 'greatness',\n 'portable',\n 'monumental',\n 'enchanting',\n 'enticingly',\n 'mighty',\n 'upgraded',\n 'excallent',\n 'helping',\n 'pamperedly',\n 'gems',\n 'everlasting',\n 'soulful',\n 'spellbindingly',\n 'eases',\n 'spectacular',\n 'qualify',\n 'honored',\n 'darling',\n 'champion',\n 'unity',\n 'gratifying',\n 'advocate',\n 'encouragingly',\n 'responsive',\n 'saintliness',\n 'breakthroughs',\n 'booming',\n 'amusingly',\n 'excitement',\n 'fragrant',\n 'admirer',\n 'entice',\n 'flawless',\n 'fortunately',\n 'mightily',\n 'celebrated',\n 'benefits',\n 'amusing',\n 'wins',\n 'thrifty',\n 'freedom',\n 'whoa',\n 'confidence',\n 'saint',\n 'congratulations',\n 'astounded',\n 'compactly',\n 'triumph',\n 'desiring',\n 'prefered',\n 'willingly',\n 'elegance',\n 'assure',\n 'brilliances',\n 'lucky',\n 'enraptured',\n 'homage',\n 'proactive',\n 'engaging',\n 'astound',\n 'exemplary',\n 'pampers',\n 'blossom',\n 'delightful',\n 'attraction',\n 'lucrative',\n 'well-managed',\n 'energetic',\n 'miraculously',\n 'authoritative',\n 'calmness',\n 'jubilant',\n 'resourcefulness',\n 'improved',\n 'serenity',\n 'goodly',\n 'guidance',\n 'pluses',\n 'liberate',\n 'rapturously',\n 'stylish',\n 'maneuverable',\n 'glamorous',\n 'honorable',\n 'yay',\n 'spiritual',\n 'stately',\n 'best-known',\n 'trusting',\n 'accessable',\n 'gratify',\n 'instructive',\n 'satisfying',\n 'foremost',\n 'notably',\n 'fortunate',\n 'solicitous',\n 'classy',\n 'fantastically',\n 'posh',\n 'admire',\n 'lively',\n 'undisputably',\n 'recover',\n 'swankier',\n 'enchant',\n 'survivor',\n 'enjoy',\n 'rewardingly',\n 'complemented',\n 'integrated',\n 'graciousness',\n 'productive',\n 'neat',\n 'strong',\n 'effortless',\n 'gracious',\n 'breeze',\n 'radiant',\n 'quaint',\n 'rejoicing',\n 'enviously',\n 'peach',\n 'uphold',\n 'coherence',\n 'sweet',\n 'ardent',\n 'daring',\n 'gold',\n 'excitedly',\n 'playful',\n 'memorable',\n 'electrify',\n 'heroically',\n 'tenacious',\n 'reputable',\n 'revel',\n 'sociable',\n 'congratulation',\n 'divine',\n 'delicacy',\n 'trustworthiness',\n 'pepped',\n 'cleaner',\n 'enviably',\n 'believable',\n 'swank',\n 'ardor',\n 'dignity',\n 'luxurious',\n 'stimulating',\n 'spectacularly',\n 'aspiration',\n 'prudent',\n 'trusty',\n 'constructive',\n 'well-run',\n 'exhilaratingly',\n 'non-violent',\n 'rewarding',\n 'illumine',\n 'beckoning',\n 'rejuvenate',\n 'quieter',\n 'harmonize',\n 'sincerely',\n 'spotless',\n 'youthful',\n 'boundless',\n 'courageousness',\n 'dazzled',\n 'virtuously',\n 'record-setting',\n 'chic',\n 'exuberantly',\n 'intriguingly',\n 'saver',\n 'excite',\n 'impressive',\n 'stellarly',\n 'optimal',\n 'inventive',\n 'altruistically',\n 'hallowed',\n 'secure',\n 'inspire',\n 'profuse',\n 'adventurous',\n 'cushy',\n 'victory',\n 'praise',\n 'cornerstone',\n 'educated',\n 'protective',\n 'supurb',\n 'smile',\n 'keen',\n 'beautify',\n 'prettily',\n 'sporty',\n 'thumbs-up',\n 'energize',\n 'believeable',\n 'contentment',\n 'intricate',\n 'preferable',\n 'useful',\n 'comfy',\n 'wonderful',\n 'affably',\n 'unfazed',\n 'luck',\n 'inexpensive',\n 'humorous',\n 'brightest',\n 'replaceable',\n 'propitiously',\n 'gratification',\n 'effectively',\n 'mesmerizes',\n 'silent',\n 'complimentary',\n 'law-abiding',\n 'miracles',\n 'assurances',\n 'resounding',\n 'influential',\n 'unrestricted',\n 'economical',\n 'smartly',\n 'clearly',\n 'ergonomical',\n 'rightful',\n 'resourceful',\n 'recommend',\n 'favorable',\n 'confident',\n 'soundly',\n 'gainfully',\n 'noble',\n 'scenic',\n 'gleefully',\n 'perseverance',\n 'best',\n 'revelation',\n 'interesting',\n 'fondly',\n 'zenith',\n 'roomy',\n 'incredible',\n 'first-rate',\n 'sturdy',\n 'reforms',\n 'mesmerize',\n 'providence',\n 'regally',\n 'euphorically',\n 'merit',\n 'dominates',\n 'upliftingly',\n 'sexy',\n 'hearten',\n 'patiently',\n 'eased',\n 'fastest',\n 'trendy',\n 'unquestionable',\n 'extraordinarily',\n 'noteworthy',\n 'effusion',\n 'stunning',\n 'assuredly',\n 'lead',\n 'marvellous',\n 'stupendously',\n 'worthwhile',\n 'appreciate',\n 'ease',\n 'individualized',\n 'eminent',\n 'fastest-growing',\n 'brilliantly',\n 'affordable',\n 'stylishly',\n 'deginified',\n 'rich',\n 'speedily',\n 'elegantly',\n 'idolized',\n 'noiseless',\n 'subsidize',\n 'compassion',\n 'heaven',\n 'prestige',\n 'paramount',\n 'desirous',\n 'proficient',\n 'dreamland',\n 'handily',\n 'triumphant',\n 'counter-attacks',\n 'vouch',\n 'snazzy',\n 'sparkling',\n 'restored',\n 'admiring',\n 'rapid',\n 'smartest',\n 'err-free',\n 'bright',\n 'poise',\n 'attune',\n 'goodness',\n 'preferes',\n 'delicate',\n 'dumbfounded',\n 'compact',\n 'winning',\n 'glad',\n 'support',\n 'distinction',\n 'proficiently',\n 'dotingly',\n 'outstandingly',\n 'awards',\n 'affluent',\n 'pamper',\n 'patient',\n 'peace',\n 'suitable',\n 'supporter',\n 'originality',\n 'robust',\n 'novelty',\n 'flourishing',\n 'ingenuity',\n 'affable',\n 'masterful',\n 'shimmering',\n 'sumptuous',\n 'refunded',\n 'excellence',\n 'sufficiently',\n 'tempt',\n 'plush',\n 'profound',\n 'tougher',\n 'enticing',\n 'jubilate',\n 'ecenomical',\n 'easiest',\n 'awarded',\n 'palatial',\n 'godlike',\n 'distinguished',\n 'loves',\n 'overtakes',\n 'excellant',\n 'achievements',\n 'harmonious',\n 'knowledgeable',\n 'thankful',\n 'beneficiary',\n 'truthfulness',\n 'pamperedness',\n 'adaptive',\n 'concise',\n 'rockstar',\n 'lucid',\n 'reliably',\n 'extol',\n 'straighten',\n 'endorses',\n 'responsibly',\n 'eager',\n 'beloved',\n 'entertains',\n 'timely',\n 'morality',\n 'entrancing',\n 'comely',\n 'celebrate',\n 'flatter',\n 'hotcakes',\n 'intelligence',\n 'sprightly',\n 'celebration',\n 'upheld',\n 'purposeful',\n 'prudently',\n 'kindliness',\n 'fancy',\n 'bonus',\n 'dote',\n 'compliment',\n 'matchless',\n 'outdo',\n 'bless',\n 'fair',\n 'admirable',\n 'convincingly',\n 'succes',\n 'uplifting',\n 'mesmerizing',\n 'agilely',\n 'blithe',\n 'accomplish',\n 'outdone',\n 'fiery',\n 'acclamation',\n 'led',\n 'gratifyingly',\n 'flatteringly',\n 'better',\n 'illuminati',\n 'intrigue',\n 'valor',\n 'efficiently',\n 'simplifying',\n 'roomier',\n 'togetherness',\n ...}","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[11]: {'uncomplicated',\n 'pain-free',\n 'impressed',\n 'afford',\n 'afordable',\n 'gained',\n 'positively',\n 'justly',\n 'courageously',\n 'enthralled',\n 'genuine',\n 'satisfy',\n 'fertile',\n 'sagacity',\n 'beckon',\n 'firmer',\n 'luster',\n 'invaluablely',\n 'right',\n 'endearing',\n 'fame',\n 'plusses',\n 'titillatingly',\n 'vivid',\n 'faithful',\n 'gush',\n 'improvements',\n 'consistently',\n 'beneficial',\n 'ideal',\n 'inspiring',\n 'enjoyable',\n 'endorsed',\n 'futurestic',\n 'charmingly',\n 'rejoicingly',\n 'sufficed',\n 'charisma',\n 'wisely',\n 'favored',\n 'equitable',\n 'ingenious',\n 'dead-cheap',\n 'spellbinding',\n 'coherent',\n 'exhilarate',\n 'grin',\n 'prosper',\n 'benifits',\n 'gorgeously',\n 'successes',\n 'commodious',\n 'fervid',\n 'adventuresome',\n 'excited',\n 'intelligible',\n 'irresistibly',\n 'dignify',\n 'hardier',\n 'openly',\n 'politeness',\n 'reforming',\n 'preeminent',\n 'supporting',\n 'ample',\n 'honesty',\n 'leads',\n 'illustrious',\n 'obtainable',\n 'generously',\n 'proud',\n 'piety',\n 'satisfied',\n 'painlessly',\n 'gloriously',\n 'covenant',\n 'angelic',\n 'prodigy',\n 'abounds',\n 'impartiality',\n 'overtaking',\n 'decency',\n 'stirringly',\n 'diligence',\n 'enthuse',\n 'quicker',\n 'delight',\n 'chivalry',\n 'awesomeness',\n 'jovial',\n 'whoooa',\n 'immense',\n 'available',\n 'honest',\n 'winnable',\n 'prestigious',\n 'acclaim',\n 'mastery',\n 'futuristic',\n 'properly',\n 'better-known',\n 'dominated',\n 'adroitly',\n 'shimmeringly',\n 'fantastic',\n 'unwavering',\n 'feasible',\n 'sagely',\n 'beckons',\n 'refund',\n 'high-quality',\n 'hilarious',\n 'chivalrous',\n 'gratitude',\n 'prominent',\n 'remedy',\n 'inspirational',\n 'awestruck',\n 'fave',\n 'enterprising',\n 'warmth',\n 'well-behaved',\n 'affirm',\n 'swankiest',\n 'revival',\n 'low-price',\n 'earnest',\n 'thrift',\n 'appeal',\n 'fanfare',\n 'felicitate',\n 'bloom',\n 'favorited',\n 'rightly',\n 'dexterously',\n 'sturdier',\n 'user-friendly',\n 'beckoned',\n 'ovation',\n 'gaiety',\n 'valiantly',\n 'freed',\n 'remarkably',\n 'strikingly',\n 'stupendous',\n 'surpass',\n 'easygoing',\n 'truthful',\n 'well-wishers',\n 'humble',\n 'foolproof',\n 'rational',\n 'hottest',\n 'streamlined',\n 'brisk',\n 'luckiest',\n 'fav',\n 'enchantingly',\n 'elevate',\n 'destiny',\n 'exalted',\n 'subsidizes',\n 'polished',\n 'consistent',\n 'realistic',\n 'hopeful',\n 'classic',\n 'cuteness',\n 'overtook',\n 'truthfully',\n 'peaceable',\n 'sensibly',\n 'substantive',\n 'steadiness',\n 'insightfully',\n 'glitter',\n 'appreciated',\n 'magical',\n 'razor-sharp',\n 'faith',\n 'congratulatory',\n 'energy-saving',\n 'reasonably',\n 'vigilant',\n 'pleasantly',\n 'stimulates',\n 'fine-looking',\n 'fast-paced',\n 'frugal',\n 'righteous',\n 'benevolent',\n 'unquestionably',\n 'meritorious',\n 'prosperous',\n 'well-balanced',\n 'fidelity',\n 'leading',\n 'vibrantly',\n 'dazzle',\n 'affability',\n 'pride',\n 'exultant',\n 'geeky',\n 'halcyon',\n 'tranquility',\n 'outperform',\n 'splendor',\n 'freedoms',\n 'liking',\n 'virtue',\n 'soundness',\n 'smiles',\n 'supreme',\n 'worked',\n 'cohere',\n 'reformed',\n 'important',\n 'thrillingly',\n 'stunned',\n 'approve',\n 'promise',\n 'amazing',\n 'cleanest',\n 'flashy',\n 'superiority',\n 'dead-on',\n 'remission',\n 'evocative',\n 'seamless',\n 'accessible',\n 'standout',\n 'sustainability',\n 'effusive',\n 'festive',\n 'delightfulness',\n 'gainful',\n 'unforgettable',\n 'reassurance',\n 'thrive',\n 'amiabily',\n 'outwit',\n 'relent',\n 'gleeful',\n 'champ',\n 'venerate',\n 'savior',\n 'advantage',\n 'raptureous',\n 'enjoyed',\n 'breakthrough',\n 'long-lasting',\n 'simplify',\n 'top',\n 'appreciable',\n 'handsome',\n 'swift',\n 'capable',\n 'proper',\n 'gallant',\n 'inviolate',\n 'adored',\n 'supple',\n 'promoter',\n 'sharpest',\n 'superbly',\n 'dependably',\n 'accomplishments',\n 'merciful',\n 'gracefully',\n 'convience',\n 'enthusiastically',\n 'examplary',\n 'revere',\n 'hardy',\n 'happier',\n 'fine',\n 'soothe',\n 'lean',\n 'indulgence',\n 'excitingly',\n 'eyecatching',\n 'thrilled',\n 'appreciative',\n 'awesomely',\n 'magnificence',\n 'majestic',\n 'elation',\n 'variety',\n 'inviolable',\n 'smoother',\n 'cooperative',\n 'sophisticated',\n 'shiny',\n 'hallmarks',\n 'adoringly',\n 'intriguing',\n 'skilled',\n 'refinement',\n 'loveliness',\n 'recommendations',\n 'ecstatically',\n 'mind-blowing',\n 'inestimable',\n 'achievement',\n 'viewable',\n 'evenly',\n 'sublime',\n 'liberty',\n 'danke',\n 'top-notch',\n 'contrasty',\n 'renowned',\n 'gladden',\n 'faultless',\n 'unrivaled',\n 'prudence',\n 'inspiration',\n 'winner',\n 'skillfully',\n 'admirably',\n 'dashing',\n 'dexterous',\n 'god-send',\n 'perfection',\n 'win',\n 'clever',\n 'wholeheartedly',\n 'slammin',\n 'spirited',\n 'peerless',\n 'eloquence',\n 'amazes',\n 'effective',\n 'boom',\n 'gratifies',\n 'enhances',\n 'credence',\n 'earnestness',\n 'striking',\n 'deserving',\n 'friendliness',\n 'motivated',\n 'gusto',\n 'helpful',\n 'harmless',\n 'gladness',\n 'bolster',\n 'idyllic',\n 'encourage',\n 'heal',\n 'ilu',\n 'jubilantly',\n 'brave',\n 'readable',\n 'healthy',\n 'progressive',\n 'regard',\n 'wowing',\n 'innovative',\n 'wowed',\n 'good',\n 'facilitate',\n 'entrust',\n 'immaculate',\n 'unlimited',\n 'nicest',\n 'impartial',\n 'improvement',\n 'fervent',\n 'simplifies',\n 'impeccable',\n 'hands-down',\n 'useable',\n 'exceeded',\n 'fans',\n 'pampered',\n 'tidy',\n 'satisified',\n 'calming',\n 'successful',\n 'attractively',\n 'innocuous',\n 'bountiful',\n 'rosy',\n 'multi-purpose',\n 'smoothly',\n 'savings',\n 'credible',\n 'resplendent',\n 'carefree',\n 'excelled',\n 'master',\n 'euphoric',\n 'generous',\n 'perfect',\n 'irresistible',\n 'assurance',\n 'stimulate',\n 'gem',\n 'passionate',\n 'humane',\n 'competitive',\n 'suave',\n 'well-connected',\n 'world-famous',\n 'enjoyment',\n 'glisten',\n 'self-satisfaction',\n 'sweetness',\n 'sensational',\n 'civilize',\n 'complement',\n 'fascinate',\n 'enrapt',\n 'accolades',\n 'exaltingly',\n 'gains',\n 'windfall',\n 'industrious',\n 'well-made',\n 'respectful',\n 'understandable',\n 'eagerly',\n 'eyecatch',\n 'fortitude',\n 'revolutionary',\n 'splendidly',\n 'efficacious',\n 'astonishingly',\n 'willing',\n 'famed',\n 'inpressed',\n 'user-replaceable',\n 'luxuriously',\n 'exquisite',\n 'righten',\n 'outperforms',\n 'amicably',\n 'righteousness',\n 'stellar',\n 'skillful',\n 'coolest',\n 'adulatory',\n 'suffices',\n 'prowess',\n 'fearless',\n 'decisive',\n 'positives',\n 'flutter',\n 'enliven',\n 'beneficent',\n 'exuberance',\n 'topnotch',\n 'adaptable',\n 'maturely',\n 'well-known',\n 'ecstasies',\n 'charitable',\n 'renown',\n 'commendably',\n 'favour',\n 'restructuring',\n 'modest',\n 'renaissance',\n 'amiable',\n 'courage',\n 'tenderly',\n 'reverent',\n 'magic',\n 'plentiful',\n 'shine',\n 'pepping',\n 'insightful',\n 'well-positioned',\n 'richness',\n 'thoughtfully',\n 'excel',\n 'conciliatory',\n 'tingle',\n 'fervidly',\n 'astounding',\n 'prodigious',\n 'issue-free',\n 'lawful',\n 'beautifully',\n 'restructure',\n 'lionhearted',\n 'jollify',\n 'sensitive',\n 'devout',\n 'cherub',\n 'attentive',\n 'honoring',\n 'trumpet',\n 'warmhearted',\n 'simplest',\n 'suffice',\n 'ambitious',\n 'cost-effective',\n 'sweetly',\n 'self-determination',\n 'overture',\n 'wise',\n 'refined',\n 'undaunted',\n 'work',\n 'wondrous',\n 'mature',\n 'balanced',\n 'steadfast',\n 'solidarity',\n 'pretty',\n 'cleared',\n 'contribution',\n 'neatest',\n 'profusion',\n 'non-violence',\n 'radiance',\n 'lovely',\n 'exhilaration',\n 'amenity',\n 'gaily',\n 'smitten',\n 'wows',\n 'rightness',\n 'commendable',\n 'undisputed',\n 'pleasurably',\n 'easiness',\n 'cool',\n 'conveniently',\n 'talented',\n 'bonuses',\n 'convient',\n 'effusiveness',\n 'exceeds',\n 'commend',\n 'warmly',\n 'poignant',\n 'glistening',\n 'reward',\n 'works',\n 'stabilize',\n 'elegant',\n 'lighter',\n 'prodigiously',\n 'stimulative',\n 'favor',\n 'galore',\n 'ennoble',\n 'wonderous',\n 'handsomely',\n 'super',\n 'enviable',\n 'greatest',\n 'admiringly',\n 'self-sufficient',\n 'astoundingly',\n 'leverage',\n 'outperformed',\n 'appreciatively',\n 'legendary',\n 'congenial',\n 'panoramic',\n 'exalting',\n 'thank',\n 'affection',\n 'fascinatingly',\n 'pure',\n 'avid',\n 'twinkly',\n 'fortune',\n 'clearer',\n 'decisiveness',\n 'sharper',\n 'unreal',\n 'undamaged',\n 'ebullience',\n 'greatness',\n 'portable',\n 'monumental',\n 'enchanting',\n 'enticingly',\n 'mighty',\n 'upgraded',\n 'excallent',\n 'helping',\n 'pamperedly',\n 'gems',\n 'everlasting',\n 'soulful',\n 'spellbindingly',\n 'eases',\n 'spectacular',\n 'qualify',\n 'honored',\n 'darling',\n 'champion',\n 'unity',\n 'gratifying',\n 'advocate',\n 'encouragingly',\n 'responsive',\n 'saintliness',\n 'breakthroughs',\n 'booming',\n 'amusingly',\n 'excitement',\n 'fragrant',\n 'admirer',\n 'entice',\n 'flawless',\n 'fortunately',\n 'mightily',\n 'celebrated',\n 'benefits',\n 'amusing',\n 'wins',\n 'thrifty',\n 'freedom',\n 'whoa',\n 'confidence',\n 'saint',\n 'congratulations',\n 'astounded',\n 'compactly',\n 'triumph',\n 'desiring',\n 'prefered',\n 'willingly',\n 'elegance',\n 'assure',\n 'brilliances',\n 'lucky',\n 'enraptured',\n 'homage',\n 'proactive',\n 'engaging',\n 'astound',\n 'exemplary',\n 'pampers',\n 'blossom',\n 'delightful',\n 'attraction',\n 'lucrative',\n 'well-managed',\n 'energetic',\n 'miraculously',\n 'authoritative',\n 'calmness',\n 'jubilant',\n 'resourcefulness',\n 'improved',\n 'serenity',\n 'goodly',\n 'guidance',\n 'pluses',\n 'liberate',\n 'rapturously',\n 'stylish',\n 'maneuverable',\n 'glamorous',\n 'honorable',\n 'yay',\n 'spiritual',\n 'stately',\n 'best-known',\n 'trusting',\n 'accessable',\n 'gratify',\n 'instructive',\n 'satisfying',\n 'foremost',\n 'notably',\n 'fortunate',\n 'solicitous',\n 'classy',\n 'fantastically',\n 'posh',\n 'admire',\n 'lively',\n 'undisputably',\n 'recover',\n 'swankier',\n 'enchant',\n 'survivor',\n 'enjoy',\n 'rewardingly',\n 'complemented',\n 'integrated',\n 'graciousness',\n 'productive',\n 'neat',\n 'strong',\n 'effortless',\n 'gracious',\n 'breeze',\n 'radiant',\n 'quaint',\n 'rejoicing',\n 'enviously',\n 'peach',\n 'uphold',\n 'coherence',\n 'sweet',\n 'ardent',\n 'daring',\n 'gold',\n 'excitedly',\n 'playful',\n 'memorable',\n 'electrify',\n 'heroically',\n 'tenacious',\n 'reputable',\n 'revel',\n 'sociable',\n 'congratulation',\n 'divine',\n 'delicacy',\n 'trustworthiness',\n 'pepped',\n 'cleaner',\n 'enviably',\n 'believable',\n 'swank',\n 'ardor',\n 'dignity',\n 'luxurious',\n 'stimulating',\n 'spectacularly',\n 'aspiration',\n 'prudent',\n 'trusty',\n 'constructive',\n 'well-run',\n 'exhilaratingly',\n 'non-violent',\n 'rewarding',\n 'illumine',\n 'beckoning',\n 'rejuvenate',\n 'quieter',\n 'harmonize',\n 'sincerely',\n 'spotless',\n 'youthful',\n 'boundless',\n 'courageousness',\n 'dazzled',\n 'virtuously',\n 'record-setting',\n 'chic',\n 'exuberantly',\n 'intriguingly',\n 'saver',\n 'excite',\n 'impressive',\n 'stellarly',\n 'optimal',\n 'inventive',\n 'altruistically',\n 'hallowed',\n 'secure',\n 'inspire',\n 'profuse',\n 'adventurous',\n 'cushy',\n 'victory',\n 'praise',\n 'cornerstone',\n 'educated',\n 'protective',\n 'supurb',\n 'smile',\n 'keen',\n 'beautify',\n 'prettily',\n 'sporty',\n 'thumbs-up',\n 'energize',\n 'believeable',\n 'contentment',\n 'intricate',\n 'preferable',\n 'useful',\n 'comfy',\n 'wonderful',\n 'affably',\n 'unfazed',\n 'luck',\n 'inexpensive',\n 'humorous',\n 'brightest',\n 'replaceable',\n 'propitiously',\n 'gratification',\n 'effectively',\n 'mesmerizes',\n 'silent',\n 'complimentary',\n 'law-abiding',\n 'miracles',\n 'assurances',\n 'resounding',\n 'influential',\n 'unrestricted',\n 'economical',\n 'smartly',\n 'clearly',\n 'ergonomical',\n 'rightful',\n 'resourceful',\n 'recommend',\n 'favorable',\n 'confident',\n 'soundly',\n 'gainfully',\n 'noble',\n 'scenic',\n 'gleefully',\n 'perseverance',\n 'best',\n 'revelation',\n 'interesting',\n 'fondly',\n 'zenith',\n 'roomy',\n 'incredible',\n 'first-rate',\n 'sturdy',\n 'reforms',\n 'mesmerize',\n 'providence',\n 'regally',\n 'euphorically',\n 'merit',\n 'dominates',\n 'upliftingly',\n 'sexy',\n 'hearten',\n 'patiently',\n 'eased',\n 'fastest',\n 'trendy',\n 'unquestionable',\n 'extraordinarily',\n 'noteworthy',\n 'effusion',\n 'stunning',\n 'assuredly',\n 'lead',\n 'marvellous',\n 'stupendously',\n 'worthwhile',\n 'appreciate',\n 'ease',\n 'individualized',\n 'eminent',\n 'fastest-growing',\n 'brilliantly',\n 'affordable',\n 'stylishly',\n 'deginified',\n 'rich',\n 'speedily',\n 'elegantly',\n 'idolized',\n 'noiseless',\n 'subsidize',\n 'compassion',\n 'heaven',\n 'prestige',\n 'paramount',\n 'desirous',\n 'proficient',\n 'dreamland',\n 'handily',\n 'triumphant',\n 'counter-attacks',\n 'vouch',\n 'snazzy',\n 'sparkling',\n 'restored',\n 'admiring',\n 'rapid',\n 'smartest',\n 'err-free',\n 'bright',\n 'poise',\n 'attune',\n 'goodness',\n 'preferes',\n 'delicate',\n 'dumbfounded',\n 'compact',\n 'winning',\n 'glad',\n 'support',\n 'distinction',\n 'proficiently',\n 'dotingly',\n 'outstandingly',\n 'awards',\n 'affluent',\n 'pamper',\n 'patient',\n 'peace',\n 'suitable',\n 'supporter',\n 'originality',\n 'robust',\n 'novelty',\n 'flourishing',\n 'ingenuity',\n 'affable',\n 'masterful',\n 'shimmering',\n 'sumptuous',\n 'refunded',\n 'excellence',\n 'sufficiently',\n 'tempt',\n 'plush',\n 'profound',\n 'tougher',\n 'enticing',\n 'jubilate',\n 'ecenomical',\n 'easiest',\n 'awarded',\n 'palatial',\n 'godlike',\n 'distinguished',\n 'loves',\n 'overtakes',\n 'excellant',\n 'achievements',\n 'harmonious',\n 'knowledgeable',\n 'thankful',\n 'beneficiary',\n 'truthfulness',\n 'pamperedness',\n 'adaptive',\n 'concise',\n 'rockstar',\n 'lucid',\n 'reliably',\n 'extol',\n 'straighten',\n 'endorses',\n 'responsibly',\n 'eager',\n 'beloved',\n 'entertains',\n 'timely',\n 'morality',\n 'entrancing',\n 'comely',\n 'celebrate',\n 'flatter',\n 'hotcakes',\n 'intelligence',\n 'sprightly',\n 'celebration',\n 'upheld',\n 'purposeful',\n 'prudently',\n 'kindliness',\n 'fancy',\n 'bonus',\n 'dote',\n 'compliment',\n 'matchless',\n 'outdo',\n 'bless',\n 'fair',\n 'admirable',\n 'convincingly',\n 'succes',\n 'uplifting',\n 'mesmerizing',\n 'agilely',\n 'blithe',\n 'accomplish',\n 'outdone',\n 'fiery',\n 'acclamation',\n 'led',\n 'gratifyingly',\n 'flatteringly',\n 'better',\n 'illuminati',\n 'intrigue',\n 'valor',\n 'efficiently',\n 'simplifying',\n 'roomier',\n 'togetherness',\n ...}"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 8"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"1ee59d3e-c43f-4813-8ce7-cc6c45a921e3"}}},{"cell_type":"code","source":["negative_words_set = set(negativeWordsDF.select('_c0').rdd.flatMap(lambda x : x).collect())\nnegative_words_set"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"9f67e84e-8541-47cd-864c-273c7dd55196"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[15]: {'manipulate',\n 'inaction',\n 'disagreeable',\n 'illegal',\n 'mad',\n 'overzealous',\n 'sneering',\n 'denunciations',\n 'crushed',\n 'calamitous',\n 'embattled',\n 'dark',\n 'objections',\n 'overzealously',\n 'self-criticism',\n 'warning',\n 'panders',\n 'overdue',\n 'fuss',\n 'lapsed',\n 'glower',\n '2-faces',\n 'heckle',\n 'insult',\n 'botch',\n 'toxic',\n 'vehement',\n 'perilously',\n 'smouldering',\n 'bleeding',\n 'inept',\n 'remorsefully',\n 'unruly',\n 'embarrassment',\n 'execrate',\n 'lechery',\n 'worsening',\n 'inadequate',\n 'boil',\n 'aweful',\n 'fall',\n 'hefty',\n 'insubordinate',\n 'divisively',\n 'liable',\n 'fanatics',\n 'adulterate',\n 'affliction',\n 'felon',\n 'sadly',\n 'disregardful',\n 'passe',\n 'bedlam',\n 'deject',\n 'hells',\n 'paucity',\n 'monster',\n 'stigma',\n 'untimely',\n 'inefficiently',\n 'worrier',\n 'startle',\n 'stressfully',\n 'failing',\n 'iniquitous',\n 'overplay',\n 'regression',\n 'insensitive',\n 'implausible',\n 'inappropriate',\n 'compulsive',\n 'flakieness',\n 'erosion',\n 'chaos',\n 'wild',\n 'complain',\n 'racism',\n 'gnawing',\n 'kooky',\n 'spiteful',\n 'cracked',\n 'errors',\n 'dogmatic',\n 'denigrate',\n 'unhelpful',\n 'dread',\n 'divergent',\n 'unfulfilled',\n 'exhaustion',\n 'zapped',\n 'disputed',\n 'obstructed',\n 'discourteously',\n 'downfall',\n 'incompatibility',\n 'laggy',\n 'fear',\n 'choppy',\n 'mistakes',\n 'noises',\n 'leak',\n 'lame-duck',\n 'denies',\n 'moron',\n 'raked',\n 'inadequacy',\n 'misbehavior',\n 'tentative',\n 'meager',\n 'deprecate',\n 'straining',\n 'vent',\n 'problems',\n 'slow-moving',\n 'blockhead',\n 'busybody',\n 'capriciously',\n 'blackmail',\n 'jealousness',\n 'lengthy',\n 'messes',\n 'plea',\n 'tiringly',\n 'disquietingly',\n 'assault',\n 'demonizing',\n 'ulterior',\n 'infidel',\n 'ridicules',\n 'deadbeat',\n 'squeals',\n 'sunken',\n 'unqualified',\n 'picketing',\n 'hothouse',\n 'craven',\n 'scratches',\n 'defiantly',\n 'sad',\n 'annoyances',\n 'foreboding',\n 'freezes',\n 'demon',\n 'dings',\n 'berserk',\n 'condemned',\n 'problem',\n 'refuting',\n 'deter',\n 'obstructs',\n 'coercive',\n 'unconvincingly',\n 'hassles',\n 'mispronounced',\n 'haters',\n 'irrepressible',\n 'dishearten',\n 'miscalculate',\n 'volatile',\n 'terror-genic',\n 'lanky',\n 'unseemly',\n 'bloodshed',\n 'ineffective',\n 'rigidness',\n 'contradiction',\n 'frigging',\n 'tin-y',\n 'tenderness',\n 'knave',\n 'narrower',\n 'complacent',\n 'contaminating',\n 'lonesome',\n 'allegations',\n 'naively',\n 'criticisms',\n 'adulterated',\n 'hating',\n 'inflationary',\n 'stridently',\n 'inexorably',\n 'boggle',\n 'hoard',\n 'ineloquently',\n 'tarnish',\n 'unforgiving',\n 'violent',\n 'insinuate',\n 'futility',\n 'oppressiveness',\n 'layoff',\n 'malcontent',\n 'malicious',\n 'inaccurate',\n 'subjected',\n 'oddities',\n 'slooow',\n 'unuseably',\n 'hairloss',\n 'ridicule',\n 'imperil',\n 'mockingly',\n 'insubstantially',\n 'scorn',\n 'gangster',\n 'capitulate',\n 'procrastinate',\n 'backache',\n 'suspicions',\n 'tiring',\n 'demonic',\n 'unwanted',\n 'lesser-known',\n 'incorrigible',\n 'dissonantly',\n 'unsteady',\n 'farcical',\n 'patronize',\n 'extort',\n 'hegemonistic',\n 'buckle',\n 'delinquent',\n 'ache',\n 'malevolently',\n 'mishandle',\n 'starve',\n 'scandalized',\n 'despicably',\n 'thorny',\n 'irresponsibly',\n 'defamatory',\n 'feign',\n 'brutally',\n 'crashed',\n 'qualm',\n 'furor',\n 'hotheaded',\n 'immoral',\n 'cold',\n 'delusion',\n 'hazardous',\n 'lie',\n 'sags',\n 'brutalities',\n 'scorchingly',\n 'resent',\n 'uneasily',\n 'primitive',\n 'ripoff',\n 'disinclination',\n 'worthlessness',\n 'sloooow',\n 'antagonist',\n 'scarce',\n 'cutthroat',\n 'cheater',\n 'dilemma',\n 'degeneration',\n 'defects',\n 'killing',\n 'racists',\n 'discriminatory',\n 'mortifying',\n 'scant',\n 'impunity',\n 'rail',\n 'intimidate',\n 'denunciation',\n 'feckless',\n 'jeer',\n 'brainwash',\n 'unintelligible',\n 'disconsolation',\n 'flawed',\n 'mocked',\n 'self-destructive',\n 'laughable',\n 'predatory',\n 'fraught',\n 'lapse',\n 'unaccustomed',\n 'frets',\n 'throbbing',\n 'disapproving',\n 'discriminate',\n 'impulsively',\n 'offender',\n 'difficulties',\n 'inconsistent',\n 'rankle',\n 'conflicted',\n 'conspicuously',\n 'dispute',\n 'halfhearted',\n 'jutter',\n 'malice',\n 'strict',\n 'unfunded',\n 'explode',\n 'cataclysm',\n 'dissonant',\n 'over-balanced',\n 'sneaky',\n 'stink',\n 'less-developed',\n 'darker',\n 'sufferer',\n 'ill-mannered',\n 'begging',\n 'refuses',\n 'inoperable',\n 'distorted',\n 'harridan',\n 'suffocate',\n 'backbite',\n 'consternation',\n 'inescapable',\n 'snub',\n 'ignorance',\n 'apologist',\n 'cheats',\n 'combative',\n 'zombie',\n 'illegally',\n 'wiles',\n 'admonishingly',\n 'unproven',\n 'illusory',\n 'wily',\n 'bewildering',\n 'cravenly',\n 'belligerent',\n 'stuttering',\n 'puppets',\n 'restricted',\n 'strife',\n 'trivial',\n 'watered-down',\n 'arduous',\n 'outsider',\n 'picketed',\n 'decadence',\n 'commotion',\n 'excruciating',\n 'flakey',\n 'inequities',\n 'prick',\n 'refuse',\n 'unproved',\n 'unreasonable',\n 'skeptically',\n 'low-rated',\n 'fatuous',\n 'fibber',\n 'flagrant',\n 'flees',\n 'impetuous',\n 'funny',\n 'inability',\n 'impinge',\n 'intractable',\n 'irritation',\n 'interruptions',\n 'profanity',\n 'rupture',\n 'subversively',\n 'vexation',\n 'fatalistically',\n 'catastrophes',\n 'horrid',\n 'painfull',\n 'breach',\n 'ripped',\n 'shimmer',\n 'unrealistic',\n 'prejudicial',\n 'aimless',\n 'devilishly',\n 'clumsy',\n 'hesitant',\n 'hardhearted',\n 'bad',\n 'damage',\n 'na�ve',\n 'temper',\n 'exhorbitant',\n 'intolerance',\n 'retract',\n 'agonizing',\n 'condemn',\n 'inexpiable',\n 'laughingstock',\n 'snobbish',\n 'stingingly',\n 'unethical',\n 'concerns',\n 'issue',\n 'rampant',\n 'fumble',\n 'enflame',\n 'immoderate',\n 'debasement',\n 'bruises',\n 'creepy',\n 'diatribes',\n 'exhausts',\n 'disturb',\n 'strictly',\n 'unhappiness',\n 'time-consuming',\n 'damned',\n 'arbitrary',\n 'inanely',\n 'relentlessness',\n 'pervert',\n 'bungler',\n 'blockage',\n 'bigotries',\n 'despotism',\n 'disorderly',\n 'autocrat',\n 'damnably',\n 'importune',\n 'slanderously',\n 'forgetful',\n 'disapprove',\n 'uproot',\n 'overtaxed',\n 'defy',\n 'ding',\n 'sarcasm',\n 'depressions',\n 'gape',\n 'tangle',\n 'ail',\n 'desert',\n 'dishonest',\n 'deceptive',\n 'unaccessible',\n 'effigy',\n 'rumors',\n 'devious',\n 'stinging',\n 'butcher',\n 'craze',\n 'ordeal',\n 'reluctantly',\n 'slander',\n 'demise',\n 'amiss',\n 'dejected',\n 'sap',\n 'wobbles',\n 'unnatural',\n 'desecrate',\n 'brimstone',\n 'dilly-dally',\n 'disrespecting',\n 'forbidding',\n 'illness',\n 'imposers',\n 'clueless',\n 'tantrum',\n 'heresy',\n 'inadequately',\n 'aggravate',\n 'contrive',\n 'merciless',\n 'adversarial',\n 'recant',\n 'fallout',\n 'ridiculous',\n 'impermissible',\n 'cheated',\n 'jittery',\n 'anxiousness',\n 'downheartedly',\n 'deficiency',\n 'clamor',\n 'fatique',\n 'hung',\n 'smelly',\n 'evasion',\n 'murderous',\n 'preoccupy',\n 'addicting',\n 'taunting',\n 'tawdry',\n 'blasted',\n 'hype',\n 'decrepit',\n 'debauch',\n 'opportunistic',\n 'pretentiously',\n 'smut',\n 'mortify',\n 'lambast',\n 'irretating',\n 'perturb',\n 'misconceptions',\n 'cataclysmal',\n 'pity',\n 'anti-occupation',\n 'swelled',\n 'abusive',\n 'deplete',\n 'mindless',\n 'bereavement',\n 'uncomfy',\n 'quandary',\n 'rumple',\n 'discontinued',\n 'bumping',\n 'concens',\n 'nauseatingly',\n 'negate',\n 'two-faces',\n 'contradictory',\n 'grumpish',\n 'horrifys',\n 'afraid',\n 'jaundiced',\n 'clog',\n 'scams',\n 'deprived',\n 'thug',\n 'stumbled',\n 'bungling',\n 'wail',\n 'treacherously',\n 'trivialize',\n 'headaches',\n 'disrupt',\n 'dent',\n 'blaspheme',\n 'cringe',\n 'negligence',\n 'deride',\n 'dubitable',\n 'madder',\n 'cronyism',\n 'timid',\n 'glib',\n 'miff',\n 'complication',\n 'detested',\n 'aches',\n 'swelling',\n 'farcically',\n 'bleak',\n 'dupe',\n 'overweight',\n 'derogatory',\n 'hasseling',\n 'conspicuous',\n 'criminal',\n 'barbarian',\n 'lacking',\n 'relapse',\n 'opinionated',\n 'flaky',\n 'disputable',\n 'cloud',\n 'discombobulate',\n 'gimmick',\n 'raving',\n 'struggle',\n 'woe',\n 'overturn',\n 'disintegrated',\n 'treasonous',\n 'bemused',\n 'inculcate',\n 'murder',\n 'jolt',\n 'crack',\n 'adulteration',\n 'bump',\n 'antagonize',\n 'sloooooooooooooow',\n 'hysteric',\n 'uglier',\n 'strangely',\n 'terribly',\n 'diss',\n 'worthless',\n 'thoughtlessness',\n 'misbecome',\n 'far-fetched',\n 'overwhelms',\n 'nepotism',\n 'sinfully',\n 'calumny',\n 'insufferable',\n 'inconsequently',\n 'stutter',\n 'sue',\n 'undesirable',\n 'erase',\n 'flagging',\n 'misunderstood',\n 'cruelness',\n 'unease',\n 'infuriatingly',\n 'shortcomings',\n 'irrecoverable',\n 'occlude',\n 'jitter',\n 'feint',\n 'unsettle',\n 'oppress',\n 'wrought',\n 'unacceptable',\n 'over-acted',\n 'byzantine',\n 'disadvantageous',\n 'downfallen',\n 'regrettably',\n 'direly',\n 'hostilities',\n 'self-serving',\n 'vexingly',\n 'betraying',\n 'pratfall',\n 'foul',\n 'obtuse',\n 'bore',\n 'dubious',\n 'devastation',\n 'bothered',\n 'fanciful',\n 'hiss',\n 'self-humiliation',\n 'stutters',\n 'haze',\n 'bleeds',\n 'coercion',\n 'discontent',\n 'salacious',\n 'cunt',\n 'languorous',\n 'incomparably',\n 'struggling',\n 'terrible',\n 'woebegone',\n 'dislike',\n 'mourn',\n 'vomit',\n 'lumpy',\n 'wariness',\n 'goofy',\n 'deadweight',\n 'anger',\n 'haunting',\n 'gallingly',\n 'protracted',\n 'vibrate',\n 'foolhardy',\n 'indolent',\n 'exagerates',\n 'inconsistencies',\n 'demonizes',\n 'bribery',\n 'destitution',\n 'exagerated',\n 'snagging',\n 'oblique',\n 'uprising',\n 'forebodingly',\n 'irresolvable',\n 'despondence',\n 'slanderer',\n 'fury',\n 'finagle',\n 'nettlesome',\n 'unacceptablely',\n 'vexing',\n 'indulge',\n 'oddest',\n 'discrimination',\n 'noise',\n 'batty',\n 'antagonism',\n 'browbeat',\n 'itch',\n 'subservience',\n 'haphazard',\n 'imprecision',\n 'deceitfully',\n 'touts',\n 'uproarous',\n 'falter',\n 'zap',\n 'misreading',\n 'chatterbox',\n 'insincerity',\n 'capsize',\n 'bitingly',\n 'dastard',\n 'havoc',\n 'retard',\n 'superficially',\n 'weep',\n 'gross',\n 'recoil',\n 'usurp',\n 'ferocity',\n 'frantically',\n 'gauche',\n 'fainthearted',\n 'recourses',\n 'averse',\n 'regretted',\n 'unwatchable',\n 'unwise',\n 'frozen',\n 'loot',\n 'fiendish',\n 'rile',\n 'glitch',\n 'neglect',\n 'tacky',\n 'squander',\n 'worsen',\n 'overkill',\n 'lemon',\n 'bickering',\n 'tyrant',\n 'belittled',\n 'impediment',\n 'perverted',\n 'panicked',\n 'forswear',\n 'incoherent',\n 'nauseate',\n 'stumped',\n 'virus',\n 'demean',\n 'incessantly',\n 'officious',\n 'barbarically',\n 'oppose',\n 'remorse',\n 'drought',\n 'scaly',\n 'troublingly',\n 'cringes',\n 'disapointing',\n 'bloated',\n 'bitchy',\n 'dizzingly',\n 'inflict',\n 'ruining',\n 'shameful',\n 'meltdown',\n 'pessimistically',\n 'precipitous',\n 'stew',\n 'irrational',\n 'little-known',\n 'uncomfortable',\n 'incomplete',\n 'bewildered',\n 'calumniously',\n 'contemptuous',\n 'guilt',\n 'deluge',\n 'object',\n 'scratch',\n 'virulent',\n 'damper',\n 'instigator',\n 'outrageous',\n 'disappointed',\n 'blow',\n 'avarice',\n 'inadverently',\n 'pitiable',\n 'genocide',\n 'instigate',\n 'sting',\n 'unnoticed',\n 'unsound',\n 'complained',\n 'insignificance',\n 'disrespectable',\n 'venomously',\n 'sarcastically',\n 'breaking',\n 'grumpy',\n 'unleash',\n 'vindictiveness',\n 'stupidity',\n 'shortness',\n 'incredulously',\n 'breakup',\n 'impropriety',\n 'obsess',\n 'seriousness',\n 'fried',\n 'apathetic',\n 'abuse',\n 'spitefully',\n 'blinding',\n 'frustrations',\n 'impedance',\n 'extremists',\n 'contrariness',\n 'two-faced',\n 'short-lived',\n 'unfeeling',\n 'unfinished',\n 'incoherently',\n 'lawless',\n 'jitters',\n 'dreadfully',\n 'symptom',\n 'fearful',\n 'disgruntled',\n 'scoldingly',\n 'unpleasantries',\n 'deny',\n 'impossiblity',\n 'ignominy',\n 'chore',\n 'vulgar',\n 'funnily',\n 'noisier',\n 'picky',\n 'recessionary',\n 'infuriate',\n 'acerbate',\n 'accidental',\n 'limp',\n 'mournfully',\n 'depress',\n 'mangling',\n 'rollercoaster',\n 'neurotically',\n 'entangle',\n 'hum',\n 'bid-rigging',\n 'overpower',\n 'worried',\n 'worrying',\n 'dissatisfying',\n 'naughty',\n 'skeletons',\n 'diametrically',\n 'ineffectualness',\n 'cursed',\n 'cartoonish',\n 'disparagingly',\n 'pertinacious',\n 'bumped',\n 'congested',\n 'decline',\n 'falling',\n 'stressful',\n 'impudent',\n 'unhappy',\n 'oversights',\n 'absurd',\n 'distrusting',\n 'scoffingly',\n 'illusions',\n 'mislead',\n 'phobia',\n 'snarl',\n 'sugar-coat',\n 'disaster',\n 'cramped',\n 'vomited',\n 'destains',\n 'oversimplify',\n 'bewitch',\n 'banishment',\n 'irretrievable',\n 'biased',\n 'impure',\n 'nightmarish',\n 'wobbled',\n 'hoodwink',\n 'stiflingly',\n 'enraged',\n 'uneventful',\n 'unnervingly',\n 'misapprehend',\n 'tepid',\n 'worse',\n 'ugliest',\n 'dissidents',\n 'lewd',\n 'villify',\n 'douchebag',\n 'craftly',\n 'exorbitantly',\n 'savages',\n 'starvation',\n 'refusing',\n 'back-wood',\n 'misinform',\n 'irresolute',\n 'deceptively',\n 'melancholy',\n 'vibration',\n 'lone',\n 'belittling',\n 'incongruous',\n 'bragger',\n 'rejected',\n 'authoritarian',\n 'dissolute',\n 'stubborn',\n 'dismal',\n 'drunken',\n 'growl',\n 'flat-out',\n 'indiscreetly',\n 'fat',\n 'indoctrinate',\n 'insincerely',\n 'fleer',\n 'smokescreen',\n 'distressed',\n 'frazzle',\n 'furious',\n 'inclement',\n 'offend',\n 'contortions',\n 'debacle',\n 'limit',\n 'superstition',\n 'appallingly',\n 'dismissively',\n 'mangles',\n 'detestably',\n 'overheat',\n 'fabricate',\n 'despot',\n 'inconveniently',\n 'inhibit',\n 'inequitable',\n 'appal',\n 'expel',\n 'objectionable',\n 'paralize',\n 'disfavor',\n 'misaligned',\n 'thoughtless',\n 'disorganized',\n 'lugubrious',\n 'fucking',\n 'gloomy',\n 'repel',\n 'inflexible',\n 'peevish',\n 'depressed',\n 'thumbs-down',\n 'undermining',\n 'indifferent',\n 'lovelorn',\n 'nonresponsive',\n 'infiltrators',\n 'ultra-hardline',\n 'ineffectually',\n 'absent-minded',\n 'accuse',\n 'fatty',\n 'listless',\n 'sneakily',\n 'slowwww',\n 'beggar',\n 'disillusioned',\n 'fallen',\n 'wrong',\n 'flagrantly',\n 'lurking',\n 'downturns',\n 'provoke',\n 'belligerently',\n 'slumping',\n 'condescension',\n 'discouragement',\n 'incompetently',\n 'cynicism',\n 'concede',\n 'died',\n 'intoxicate',\n 'anti-israeli',\n 'second-class',\n 'languor',\n 'humiliate',\n 'wallow',\n 'madly',\n 'miserably',\n 'squealing',\n 'dungeon',\n 'derision',\n ...}","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[15]: {'manipulate',\n 'inaction',\n 'disagreeable',\n 'illegal',\n 'mad',\n 'overzealous',\n 'sneering',\n 'denunciations',\n 'crushed',\n 'calamitous',\n 'embattled',\n 'dark',\n 'objections',\n 'overzealously',\n 'self-criticism',\n 'warning',\n 'panders',\n 'overdue',\n 'fuss',\n 'lapsed',\n 'glower',\n '2-faces',\n 'heckle',\n 'insult',\n 'botch',\n 'toxic',\n 'vehement',\n 'perilously',\n 'smouldering',\n 'bleeding',\n 'inept',\n 'remorsefully',\n 'unruly',\n 'embarrassment',\n 'execrate',\n 'lechery',\n 'worsening',\n 'inadequate',\n 'boil',\n 'aweful',\n 'fall',\n 'hefty',\n 'insubordinate',\n 'divisively',\n 'liable',\n 'fanatics',\n 'adulterate',\n 'affliction',\n 'felon',\n 'sadly',\n 'disregardful',\n 'passe',\n 'bedlam',\n 'deject',\n 'hells',\n 'paucity',\n 'monster',\n 'stigma',\n 'untimely',\n 'inefficiently',\n 'worrier',\n 'startle',\n 'stressfully',\n 'failing',\n 'iniquitous',\n 'overplay',\n 'regression',\n 'insensitive',\n 'implausible',\n 'inappropriate',\n 'compulsive',\n 'flakieness',\n 'erosion',\n 'chaos',\n 'wild',\n 'complain',\n 'racism',\n 'gnawing',\n 'kooky',\n 'spiteful',\n 'cracked',\n 'errors',\n 'dogmatic',\n 'denigrate',\n 'unhelpful',\n 'dread',\n 'divergent',\n 'unfulfilled',\n 'exhaustion',\n 'zapped',\n 'disputed',\n 'obstructed',\n 'discourteously',\n 'downfall',\n 'incompatibility',\n 'laggy',\n 'fear',\n 'choppy',\n 'mistakes',\n 'noises',\n 'leak',\n 'lame-duck',\n 'denies',\n 'moron',\n 'raked',\n 'inadequacy',\n 'misbehavior',\n 'tentative',\n 'meager',\n 'deprecate',\n 'straining',\n 'vent',\n 'problems',\n 'slow-moving',\n 'blockhead',\n 'busybody',\n 'capriciously',\n 'blackmail',\n 'jealousness',\n 'lengthy',\n 'messes',\n 'plea',\n 'tiringly',\n 'disquietingly',\n 'assault',\n 'demonizing',\n 'ulterior',\n 'infidel',\n 'ridicules',\n 'deadbeat',\n 'squeals',\n 'sunken',\n 'unqualified',\n 'picketing',\n 'hothouse',\n 'craven',\n 'scratches',\n 'defiantly',\n 'sad',\n 'annoyances',\n 'foreboding',\n 'freezes',\n 'demon',\n 'dings',\n 'berserk',\n 'condemned',\n 'problem',\n 'refuting',\n 'deter',\n 'obstructs',\n 'coercive',\n 'unconvincingly',\n 'hassles',\n 'mispronounced',\n 'haters',\n 'irrepressible',\n 'dishearten',\n 'miscalculate',\n 'volatile',\n 'terror-genic',\n 'lanky',\n 'unseemly',\n 'bloodshed',\n 'ineffective',\n 'rigidness',\n 'contradiction',\n 'frigging',\n 'tin-y',\n 'tenderness',\n 'knave',\n 'narrower',\n 'complacent',\n 'contaminating',\n 'lonesome',\n 'allegations',\n 'naively',\n 'criticisms',\n 'adulterated',\n 'hating',\n 'inflationary',\n 'stridently',\n 'inexorably',\n 'boggle',\n 'hoard',\n 'ineloquently',\n 'tarnish',\n 'unforgiving',\n 'violent',\n 'insinuate',\n 'futility',\n 'oppressiveness',\n 'layoff',\n 'malcontent',\n 'malicious',\n 'inaccurate',\n 'subjected',\n 'oddities',\n 'slooow',\n 'unuseably',\n 'hairloss',\n 'ridicule',\n 'imperil',\n 'mockingly',\n 'insubstantially',\n 'scorn',\n 'gangster',\n 'capitulate',\n 'procrastinate',\n 'backache',\n 'suspicions',\n 'tiring',\n 'demonic',\n 'unwanted',\n 'lesser-known',\n 'incorrigible',\n 'dissonantly',\n 'unsteady',\n 'farcical',\n 'patronize',\n 'extort',\n 'hegemonistic',\n 'buckle',\n 'delinquent',\n 'ache',\n 'malevolently',\n 'mishandle',\n 'starve',\n 'scandalized',\n 'despicably',\n 'thorny',\n 'irresponsibly',\n 'defamatory',\n 'feign',\n 'brutally',\n 'crashed',\n 'qualm',\n 'furor',\n 'hotheaded',\n 'immoral',\n 'cold',\n 'delusion',\n 'hazardous',\n 'lie',\n 'sags',\n 'brutalities',\n 'scorchingly',\n 'resent',\n 'uneasily',\n 'primitive',\n 'ripoff',\n 'disinclination',\n 'worthlessness',\n 'sloooow',\n 'antagonist',\n 'scarce',\n 'cutthroat',\n 'cheater',\n 'dilemma',\n 'degeneration',\n 'defects',\n 'killing',\n 'racists',\n 'discriminatory',\n 'mortifying',\n 'scant',\n 'impunity',\n 'rail',\n 'intimidate',\n 'denunciation',\n 'feckless',\n 'jeer',\n 'brainwash',\n 'unintelligible',\n 'disconsolation',\n 'flawed',\n 'mocked',\n 'self-destructive',\n 'laughable',\n 'predatory',\n 'fraught',\n 'lapse',\n 'unaccustomed',\n 'frets',\n 'throbbing',\n 'disapproving',\n 'discriminate',\n 'impulsively',\n 'offender',\n 'difficulties',\n 'inconsistent',\n 'rankle',\n 'conflicted',\n 'conspicuously',\n 'dispute',\n 'halfhearted',\n 'jutter',\n 'malice',\n 'strict',\n 'unfunded',\n 'explode',\n 'cataclysm',\n 'dissonant',\n 'over-balanced',\n 'sneaky',\n 'stink',\n 'less-developed',\n 'darker',\n 'sufferer',\n 'ill-mannered',\n 'begging',\n 'refuses',\n 'inoperable',\n 'distorted',\n 'harridan',\n 'suffocate',\n 'backbite',\n 'consternation',\n 'inescapable',\n 'snub',\n 'ignorance',\n 'apologist',\n 'cheats',\n 'combative',\n 'zombie',\n 'illegally',\n 'wiles',\n 'admonishingly',\n 'unproven',\n 'illusory',\n 'wily',\n 'bewildering',\n 'cravenly',\n 'belligerent',\n 'stuttering',\n 'puppets',\n 'restricted',\n 'strife',\n 'trivial',\n 'watered-down',\n 'arduous',\n 'outsider',\n 'picketed',\n 'decadence',\n 'commotion',\n 'excruciating',\n 'flakey',\n 'inequities',\n 'prick',\n 'refuse',\n 'unproved',\n 'unreasonable',\n 'skeptically',\n 'low-rated',\n 'fatuous',\n 'fibber',\n 'flagrant',\n 'flees',\n 'impetuous',\n 'funny',\n 'inability',\n 'impinge',\n 'intractable',\n 'irritation',\n 'interruptions',\n 'profanity',\n 'rupture',\n 'subversively',\n 'vexation',\n 'fatalistically',\n 'catastrophes',\n 'horrid',\n 'painfull',\n 'breach',\n 'ripped',\n 'shimmer',\n 'unrealistic',\n 'prejudicial',\n 'aimless',\n 'devilishly',\n 'clumsy',\n 'hesitant',\n 'hardhearted',\n 'bad',\n 'damage',\n 'na�ve',\n 'temper',\n 'exhorbitant',\n 'intolerance',\n 'retract',\n 'agonizing',\n 'condemn',\n 'inexpiable',\n 'laughingstock',\n 'snobbish',\n 'stingingly',\n 'unethical',\n 'concerns',\n 'issue',\n 'rampant',\n 'fumble',\n 'enflame',\n 'immoderate',\n 'debasement',\n 'bruises',\n 'creepy',\n 'diatribes',\n 'exhausts',\n 'disturb',\n 'strictly',\n 'unhappiness',\n 'time-consuming',\n 'damned',\n 'arbitrary',\n 'inanely',\n 'relentlessness',\n 'pervert',\n 'bungler',\n 'blockage',\n 'bigotries',\n 'despotism',\n 'disorderly',\n 'autocrat',\n 'damnably',\n 'importune',\n 'slanderously',\n 'forgetful',\n 'disapprove',\n 'uproot',\n 'overtaxed',\n 'defy',\n 'ding',\n 'sarcasm',\n 'depressions',\n 'gape',\n 'tangle',\n 'ail',\n 'desert',\n 'dishonest',\n 'deceptive',\n 'unaccessible',\n 'effigy',\n 'rumors',\n 'devious',\n 'stinging',\n 'butcher',\n 'craze',\n 'ordeal',\n 'reluctantly',\n 'slander',\n 'demise',\n 'amiss',\n 'dejected',\n 'sap',\n 'wobbles',\n 'unnatural',\n 'desecrate',\n 'brimstone',\n 'dilly-dally',\n 'disrespecting',\n 'forbidding',\n 'illness',\n 'imposers',\n 'clueless',\n 'tantrum',\n 'heresy',\n 'inadequately',\n 'aggravate',\n 'contrive',\n 'merciless',\n 'adversarial',\n 'recant',\n 'fallout',\n 'ridiculous',\n 'impermissible',\n 'cheated',\n 'jittery',\n 'anxiousness',\n 'downheartedly',\n 'deficiency',\n 'clamor',\n 'fatique',\n 'hung',\n 'smelly',\n 'evasion',\n 'murderous',\n 'preoccupy',\n 'addicting',\n 'taunting',\n 'tawdry',\n 'blasted',\n 'hype',\n 'decrepit',\n 'debauch',\n 'opportunistic',\n 'pretentiously',\n 'smut',\n 'mortify',\n 'lambast',\n 'irretating',\n 'perturb',\n 'misconceptions',\n 'cataclysmal',\n 'pity',\n 'anti-occupation',\n 'swelled',\n 'abusive',\n 'deplete',\n 'mindless',\n 'bereavement',\n 'uncomfy',\n 'quandary',\n 'rumple',\n 'discontinued',\n 'bumping',\n 'concens',\n 'nauseatingly',\n 'negate',\n 'two-faces',\n 'contradictory',\n 'grumpish',\n 'horrifys',\n 'afraid',\n 'jaundiced',\n 'clog',\n 'scams',\n 'deprived',\n 'thug',\n 'stumbled',\n 'bungling',\n 'wail',\n 'treacherously',\n 'trivialize',\n 'headaches',\n 'disrupt',\n 'dent',\n 'blaspheme',\n 'cringe',\n 'negligence',\n 'deride',\n 'dubitable',\n 'madder',\n 'cronyism',\n 'timid',\n 'glib',\n 'miff',\n 'complication',\n 'detested',\n 'aches',\n 'swelling',\n 'farcically',\n 'bleak',\n 'dupe',\n 'overweight',\n 'derogatory',\n 'hasseling',\n 'conspicuous',\n 'criminal',\n 'barbarian',\n 'lacking',\n 'relapse',\n 'opinionated',\n 'flaky',\n 'disputable',\n 'cloud',\n 'discombobulate',\n 'gimmick',\n 'raving',\n 'struggle',\n 'woe',\n 'overturn',\n 'disintegrated',\n 'treasonous',\n 'bemused',\n 'inculcate',\n 'murder',\n 'jolt',\n 'crack',\n 'adulteration',\n 'bump',\n 'antagonize',\n 'sloooooooooooooow',\n 'hysteric',\n 'uglier',\n 'strangely',\n 'terribly',\n 'diss',\n 'worthless',\n 'thoughtlessness',\n 'misbecome',\n 'far-fetched',\n 'overwhelms',\n 'nepotism',\n 'sinfully',\n 'calumny',\n 'insufferable',\n 'inconsequently',\n 'stutter',\n 'sue',\n 'undesirable',\n 'erase',\n 'flagging',\n 'misunderstood',\n 'cruelness',\n 'unease',\n 'infuriatingly',\n 'shortcomings',\n 'irrecoverable',\n 'occlude',\n 'jitter',\n 'feint',\n 'unsettle',\n 'oppress',\n 'wrought',\n 'unacceptable',\n 'over-acted',\n 'byzantine',\n 'disadvantageous',\n 'downfallen',\n 'regrettably',\n 'direly',\n 'hostilities',\n 'self-serving',\n 'vexingly',\n 'betraying',\n 'pratfall',\n 'foul',\n 'obtuse',\n 'bore',\n 'dubious',\n 'devastation',\n 'bothered',\n 'fanciful',\n 'hiss',\n 'self-humiliation',\n 'stutters',\n 'haze',\n 'bleeds',\n 'coercion',\n 'discontent',\n 'salacious',\n 'cunt',\n 'languorous',\n 'incomparably',\n 'struggling',\n 'terrible',\n 'woebegone',\n 'dislike',\n 'mourn',\n 'vomit',\n 'lumpy',\n 'wariness',\n 'goofy',\n 'deadweight',\n 'anger',\n 'haunting',\n 'gallingly',\n 'protracted',\n 'vibrate',\n 'foolhardy',\n 'indolent',\n 'exagerates',\n 'inconsistencies',\n 'demonizes',\n 'bribery',\n 'destitution',\n 'exagerated',\n 'snagging',\n 'oblique',\n 'uprising',\n 'forebodingly',\n 'irresolvable',\n 'despondence',\n 'slanderer',\n 'fury',\n 'finagle',\n 'nettlesome',\n 'unacceptablely',\n 'vexing',\n 'indulge',\n 'oddest',\n 'discrimination',\n 'noise',\n 'batty',\n 'antagonism',\n 'browbeat',\n 'itch',\n 'subservience',\n 'haphazard',\n 'imprecision',\n 'deceitfully',\n 'touts',\n 'uproarous',\n 'falter',\n 'zap',\n 'misreading',\n 'chatterbox',\n 'insincerity',\n 'capsize',\n 'bitingly',\n 'dastard',\n 'havoc',\n 'retard',\n 'superficially',\n 'weep',\n 'gross',\n 'recoil',\n 'usurp',\n 'ferocity',\n 'frantically',\n 'gauche',\n 'fainthearted',\n 'recourses',\n 'averse',\n 'regretted',\n 'unwatchable',\n 'unwise',\n 'frozen',\n 'loot',\n 'fiendish',\n 'rile',\n 'glitch',\n 'neglect',\n 'tacky',\n 'squander',\n 'worsen',\n 'overkill',\n 'lemon',\n 'bickering',\n 'tyrant',\n 'belittled',\n 'impediment',\n 'perverted',\n 'panicked',\n 'forswear',\n 'incoherent',\n 'nauseate',\n 'stumped',\n 'virus',\n 'demean',\n 'incessantly',\n 'officious',\n 'barbarically',\n 'oppose',\n 'remorse',\n 'drought',\n 'scaly',\n 'troublingly',\n 'cringes',\n 'disapointing',\n 'bloated',\n 'bitchy',\n 'dizzingly',\n 'inflict',\n 'ruining',\n 'shameful',\n 'meltdown',\n 'pessimistically',\n 'precipitous',\n 'stew',\n 'irrational',\n 'little-known',\n 'uncomfortable',\n 'incomplete',\n 'bewildered',\n 'calumniously',\n 'contemptuous',\n 'guilt',\n 'deluge',\n 'object',\n 'scratch',\n 'virulent',\n 'damper',\n 'instigator',\n 'outrageous',\n 'disappointed',\n 'blow',\n 'avarice',\n 'inadverently',\n 'pitiable',\n 'genocide',\n 'instigate',\n 'sting',\n 'unnoticed',\n 'unsound',\n 'complained',\n 'insignificance',\n 'disrespectable',\n 'venomously',\n 'sarcastically',\n 'breaking',\n 'grumpy',\n 'unleash',\n 'vindictiveness',\n 'stupidity',\n 'shortness',\n 'incredulously',\n 'breakup',\n 'impropriety',\n 'obsess',\n 'seriousness',\n 'fried',\n 'apathetic',\n 'abuse',\n 'spitefully',\n 'blinding',\n 'frustrations',\n 'impedance',\n 'extremists',\n 'contrariness',\n 'two-faced',\n 'short-lived',\n 'unfeeling',\n 'unfinished',\n 'incoherently',\n 'lawless',\n 'jitters',\n 'dreadfully',\n 'symptom',\n 'fearful',\n 'disgruntled',\n 'scoldingly',\n 'unpleasantries',\n 'deny',\n 'impossiblity',\n 'ignominy',\n 'chore',\n 'vulgar',\n 'funnily',\n 'noisier',\n 'picky',\n 'recessionary',\n 'infuriate',\n 'acerbate',\n 'accidental',\n 'limp',\n 'mournfully',\n 'depress',\n 'mangling',\n 'rollercoaster',\n 'neurotically',\n 'entangle',\n 'hum',\n 'bid-rigging',\n 'overpower',\n 'worried',\n 'worrying',\n 'dissatisfying',\n 'naughty',\n 'skeletons',\n 'diametrically',\n 'ineffectualness',\n 'cursed',\n 'cartoonish',\n 'disparagingly',\n 'pertinacious',\n 'bumped',\n 'congested',\n 'decline',\n 'falling',\n 'stressful',\n 'impudent',\n 'unhappy',\n 'oversights',\n 'absurd',\n 'distrusting',\n 'scoffingly',\n 'illusions',\n 'mislead',\n 'phobia',\n 'snarl',\n 'sugar-coat',\n 'disaster',\n 'cramped',\n 'vomited',\n 'destains',\n 'oversimplify',\n 'bewitch',\n 'banishment',\n 'irretrievable',\n 'biased',\n 'impure',\n 'nightmarish',\n 'wobbled',\n 'hoodwink',\n 'stiflingly',\n 'enraged',\n 'uneventful',\n 'unnervingly',\n 'misapprehend',\n 'tepid',\n 'worse',\n 'ugliest',\n 'dissidents',\n 'lewd',\n 'villify',\n 'douchebag',\n 'craftly',\n 'exorbitantly',\n 'savages',\n 'starvation',\n 'refusing',\n 'back-wood',\n 'misinform',\n 'irresolute',\n 'deceptively',\n 'melancholy',\n 'vibration',\n 'lone',\n 'belittling',\n 'incongruous',\n 'bragger',\n 'rejected',\n 'authoritarian',\n 'dissolute',\n 'stubborn',\n 'dismal',\n 'drunken',\n 'growl',\n 'flat-out',\n 'indiscreetly',\n 'fat',\n 'indoctrinate',\n 'insincerely',\n 'fleer',\n 'smokescreen',\n 'distressed',\n 'frazzle',\n 'furious',\n 'inclement',\n 'offend',\n 'contortions',\n 'debacle',\n 'limit',\n 'superstition',\n 'appallingly',\n 'dismissively',\n 'mangles',\n 'detestably',\n 'overheat',\n 'fabricate',\n 'despot',\n 'inconveniently',\n 'inhibit',\n 'inequitable',\n 'appal',\n 'expel',\n 'objectionable',\n 'paralize',\n 'disfavor',\n 'misaligned',\n 'thoughtless',\n 'disorganized',\n 'lugubrious',\n 'fucking',\n 'gloomy',\n 'repel',\n 'inflexible',\n 'peevish',\n 'depressed',\n 'thumbs-down',\n 'undermining',\n 'indifferent',\n 'lovelorn',\n 'nonresponsive',\n 'infiltrators',\n 'ultra-hardline',\n 'ineffectually',\n 'absent-minded',\n 'accuse',\n 'fatty',\n 'listless',\n 'sneakily',\n 'slowwww',\n 'beggar',\n 'disillusioned',\n 'fallen',\n 'wrong',\n 'flagrantly',\n 'lurking',\n 'downturns',\n 'provoke',\n 'belligerently',\n 'slumping',\n 'condescension',\n 'discouragement',\n 'incompetently',\n 'cynicism',\n 'concede',\n 'died',\n 'intoxicate',\n 'anti-israeli',\n 'second-class',\n 'languor',\n 'humiliate',\n 'wallow',\n 'madly',\n 'miserably',\n 'squealing',\n 'dungeon',\n 'derision',\n ...}"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 9"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"7d984ea7-322d-4b31-8a44-400aad77d4d3"}}},{"cell_type":"code","source":["list_of_words = wordPairRDD.flatMap(lambda x: x[1])\nlist_of_words.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"ef40edc2-ec7f-4407-8ba3-23d8525c06c5"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[16]: ['Next-generation', 'sequencing', 'is', 'increasingly', 'being']","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[16]: ['Next-generation', 'sequencing', 'is', 'increasingly', 'being']"]}}],"execution_count":0},{"cell_type":"code","source":["posNegPairRDD = wordPairRDD.map(lambda x: (x[0], len(list(set(x[1]) & positive_words_set)), len(list(set(x[1]) & negative_words_set))))"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"8eabea15-c445-462a-80d5-09a136a68536"}},"outputs":[],"execution_count":0},{"cell_type":"code","source":["posNegPairRDD.take(10)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"f39cae93-194e-4a81-bf99-831eea893141"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[18]: [('vho70jcx', 8, 0),\n ('i9tbix2v', 3, 2),\n ('62gfisc6', 3, 2),\n ('058r9486', 5, 4),\n ('wich35l7', 7, 6),\n ('z3tgnzth', 2, 5),\n ('1xxrnpg3', 3, 6),\n ('8ilzm51q', 3, 3),\n ('wafvnbdu', 2, 2),\n ('4xocqn6o', 6, 2)]","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[18]: [('vho70jcx', 8, 0),\n ('i9tbix2v', 3, 2),\n ('62gfisc6', 3, 2),\n ('058r9486', 5, 4),\n ('wich35l7', 7, 6),\n ('z3tgnzth', 2, 5),\n ('1xxrnpg3', 3, 6),\n ('8ilzm51q', 3, 3),\n ('wafvnbdu', 2, 2),\n ('4xocqn6o', 6, 2)]"]}}],"execution_count":0},{"cell_type":"markdown","source":["### Number 10"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"a2b39a66-8ea3-464e-b548-2771e4ba1d3a"}}},{"cell_type":"code","source":["positiveSortRDD = posNegPairRDD.sortBy(lambda z: z[1], ascending=False)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"3147ca88-71c2-4aca-b11f-fc7217b2f48a"}},"outputs":[],"execution_count":0},{"cell_type":"code","source":["positiveSortRDD.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"eb20a58c-a259-418d-896d-14e1bc9dad96"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[21]: [('1vimqhdp', 55, 83),\n ('9wh2ahst', 29, 19),\n ('tuhesrpb', 28, 39),\n ('qkfxfmxb', 27, 7),\n ('0y122bxf', 18, 15)]","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[21]: [('1vimqhdp', 55, 83),\n ('9wh2ahst', 29, 19),\n ('tuhesrpb', 28, 39),\n ('qkfxfmxb', 27, 7),\n ('0y122bxf', 18, 15)]"]}}],"execution_count":0},{"cell_type":"code","source":["negativeSortRDD = posNegPairRDD.sortBy(lambda z: z[2], ascending=False)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"bd7eab60-56b4-406a-a06f-fa44d59aae8d"}},"outputs":[],"execution_count":0},{"cell_type":"code","source":["negativeSortRDD.take(5)"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"e34456ab-c0ff-4c1b-a050-b1cdb8c74f9d"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[23]: [('1vimqhdp', 55, 83),\n ('5y1qgjr8', 16, 40),\n ('tuhesrpb', 28, 39),\n ('ij3ncdb6', 13, 35),\n ('gvh0wdxn', 10, 25)]","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[23]: [('1vimqhdp', 55, 83),\n ('5y1qgjr8', 16, 40),\n ('tuhesrpb', 28, 39),\n ('ij3ncdb6', 13, 35),\n ('gvh0wdxn', 10, 25)]"]}}],"execution_count":0},{"cell_type":"code","source":["compairRDD = positiveSortRDD.first() == negativeSortRDD.first()\ncompairRDD"],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"eacf3962-c9e5-4fe6-a7f7-53e22a419478"}},"outputs":[{"output_type":"display_data","metadata":{"application/vnd.databricks.v1+output":{"datasetInfos":[],"data":"Out[24]: True","removedWidgets":[],"addedWidgets":{},"metadata":{},"type":"ansi","arguments":{}}},"output_type":"display_data","data":{"text/plain":["Out[24]: True"]}}],"execution_count":0},{"cell_type":"code","source":[""],"metadata":{"application/vnd.databricks.v1+cell":{"title":"","showTitle":false,"inputWidgets":{},"nuid":"ebeebf63-8805-431e-8e79-b6bb3423b9ca"}},"outputs":[],"execution_count":0}],"metadata":{"application/vnd.databricks.v1+notebook":{"notebookName":"sentiment-analysis","dashboards":[],"notebookMetadata":{"pythonIndentUnit":4},"language":"python","widgets":{},"notebookOrigID":1702114007074515}},"nbformat":4,"nbformat_minor":0}
